<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <title>Enumerated value concepts for application building</title>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
    <link href="http://www.diversitycampus.net/Workbench02.CSS" type="text/css" rel="stylesheet" />
    <link href="http://www.diversitycampus.net/Workbench02p.CSS" type="text/css" rel="stylesheet" media="print" />
  </head>
  <body lang="en-us" xml:lang="en-us">
    <h1>Enumerated value concepts for application building</h1>
    <p>This document is generated by an xslt script from the enumerations present in the following schema: "Unified Biosciences Information Framework (UBIF) 1.1". Enumerations are converted to SDD descriptive concepts (enumerated values are represented by concept states). The html report generated for these values / concept states is intended for documentation and to improve discussion and correction of errors (please comment on http://wiki.tdwg.org/twiki/bin/view/UBIF/EnumeratedValues). The xml representation follows the general conventions of UBIF documents and may be easier to import or integrate into user interfaces than the schema itself. This is especially true for SDD documents, where a large part of the terminology is provided in the form of data documents by the experts of a given knowledge domain. This document can be used side-by-side as schema-defined terminology with user-defined terminology.</p>
    <h2>
      <a name="toc">Concepts documented here:</a>
    </h2>
    <ul class="compact" compact="compact">
      <li>
        <a href="#RevisionStatusEnum">RevisionStatus</a>
      </li>
      <li>
        <a href="#ExpertiseLevelEnum">ExpertiseLevel</a>
      </li>
      <li>
        <a href="#ResourceTypeEnum">ResourceType</a>
      </li>
      <li>
        <a href="#TelephoneDeviceEnum">TelephoneDevice</a>
      </li>
      <li>
        <a href="#Rating1to5Enum">Rating1to5</a>
      </li>
      <li>
        <a href="#TextDirectionalityEnum">TextDirectionality</a>
      </li>
      <li>
        <a href="#StringFormattingTypeEnum">StringFormattingType</a>
      </li>
      <li>
        <a href="#MeasurementUnitPrefixEnum">MeasurementUnitPrefix</a>
      </li>
      <li>
        <a href="#StatisticalMeasurementScaleEnum">StatisticalMeasurementScale</a>
      </li>
      <li>
        <a href="#QuantitativeMeasurementScaleEnum">QuantitativeMeasurementScale</a>
      </li>
      <li>
        <a href="#CategoricalMeasurementScaleEnum">CategoricalMeasurementScale</a>
      </li>
      <li>
        <a href="#DataStatusEnum">DataStatus</a>
      </li>
      <li>
        <a href="#UnivarAnyStatMeasureEnum">UnivarAnyStatMeasure</a>
      </li>
      <li>
        <a href="#UnivarSimpleStatMeasureEnum">UnivarSimpleStatMeasure</a>
      </li>
      <li>
        <a href="#UnivarParamStatMeasureEnum">UnivarParamStatMeasure</a>
      </li>
      <li>
        <a href="#AgentRoleEnum">AgentRole</a>
      </li>
      <li>
        <a href="#AgentCreatorContribRoleEnum">AgentCreatorContribRole</a>
      </li>
      <li>
        <a href="#AgentCreatorRoleEnum">AgentCreatorRole</a>
      </li>
      <li>
        <a href="#AgentContributorRoleEnum">AgentContributorRole</a>
      </li>
      <li>
        <a href="#AgentOwnerRoleEnum">AgentOwnerRole</a>
      </li>
      <li>
        <a href="#LabelRoleEnum">LabelRole</a>
      </li>
      <li>
        <a href="#DetailRoleEnum">DetailRole</a>
      </li>
      <li>
        <a href="#MediaRepresentationRoleEnum">MediaRepresentationRole</a>
      </li>
      <li>
        <a href="#LinkingRelEnum">LinkingRel</a>
      </li>
      <li>
        <a href="#IRPStatementRoleEnum">IRPStatementRole</a>
      </li>
      <li>
        <a href="#SexStatusEnum">SexStatus</a>
      </li>
      <li>
        <a href="#BasicSexStatusEnum">BasicSexStatus</a>
      </li>
      <li>
        <a href="#AdditionalSexStatusEnum">AdditionalSexStatus</a>
      </li>
      <li>
        <a href="#TaxonHierarchyTypeEnum">TaxonHierarchyType</a>
      </li>
      <li>
        <a href="#IdentificationCertaintyEnum">IdentificationCertainty</a>
      </li>
      <li>
        <a href="#NomenclaturalTypeStatusOfSpecimensEnum">NomenclaturalTypeStatusOfSpecimens</a>
      </li>
      <li>
        <a href="#NomenclaturalStatusEnum">NomenclaturalStatus</a>
      </li>
      <li>
        <a href="#NomenclaturalCodesEnum">NomenclaturalCodes</a>
      </li>
      <li>
        <a href="#TaxonomicRankEnum">TaxonomicRank</a>
      </li>
      <li>
        <a href="#TaxonomicRankBelowSubspeciesEnum">TaxonomicRankBelowSubspecies</a>
      </li>
      <li>
        <a href="#TaxonomicRankSpeciesGroupEnum">TaxonomicRankSpeciesGroup</a>
      </li>
      <li>
        <a href="#TaxonomicRankGenusSubdivisionEnum">TaxonomicRankGenusSubdivision</a>
      </li>
      <li>
        <a href="#TaxonomicRankGenusGroupEnum">TaxonomicRankGenusGroup</a>
      </li>
      <li>
        <a href="#TaxonomicRankFamilySubdivisionEnum">TaxonomicRankFamilySubdivision</a>
      </li>
      <li>
        <a href="#TaxonomicRankFamilyGroupEnum">TaxonomicRankFamilyGroup</a>
      </li>
      <li>
        <a href="#TaxonomicRankAboveSuperfamilyEnum">TaxonomicRankAboveSuperfamily</a>
      </li>
    </ul>
    <p> </p>
    <hr />
    <h3>
      <a name="RevisionStatusEnum" id="RevisionStatusEnum">RevisionStatus</a>
    </h3>
    <p>Controlled vocabulary expressing the revision status as assessed by creators or editors or objects. It may apply to an entire data set as well as to individual objects (a specimen, a taxon name, a description, etc.). Exact semantics are defined only for the first and the last two categories. The semantics of the intermediate levels (1 to 5) may be chosen freely by the user (and associated with the actual, project dependend workflow, e.g. 'draft', 'review', 'approved', 'stable'). The added semantics should, however, conserve the order of revision status values. If, for example, three revision steps are planned (2 intermediate, reaching FullyRevised on third), it is recommended to use RevisionLevel2, RevisionLevel4, FullyRevised.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Unrevised</td>
          <td>Unrevised</td>
          <td> </td>
          <td>The data have been input, but no separate revision was performed.</td>
        </tr>
        <tr class="data2">
          <td>RevisionLevel1</td>
          <td>Revision level 1 of 5</td>
          <td> </td>
          <td>For example, in a collection less than ca. 20 % of the data are revised, or on a single object only a plausibility check has been performed.</td>
        </tr>
        <tr class="data1">
          <td>RevisionLevel2</td>
          <td>Revision level 2 of 5</td>
          <td> </td>
          <td>For example, in a collection ca. 41-60 % of the data are revised, or on a single object the data are compared carefully with the source.</td>
        </tr>
        <tr class="data2">
          <td>RevisionLevel3</td>
          <td>Revision level 3 of 5</td>
          <td> </td>
          <td>For example, ca. 41-60 % of the data are revised.</td>
        </tr>
        <tr class="data1">
          <td>RevisionLevel4</td>
          <td>Revision level 4 of 5</td>
          <td> </td>
          <td>For example, ca. 61-80 % of the data are revised.</td>
        </tr>
        <tr class="data2">
          <td>RevisionLevel5</td>
          <td>Revision level 5 of 5</td>
          <td> </td>
          <td>For example, more than 80% revised (but not yet completed).</td>
        </tr>
        <tr class="data1">
          <td>FullyRevised</td>
          <td>Revision completed</td>
          <td> </td>
          <td>The data are completely revised given the available time and the goals set for the project. This does not necessarily imply that the data are complete and perfectly revised in a scientific sense.</td>
        </tr>
        <tr class="data2">
          <td>Deprecated</td>
          <td>Deprecated</td>
          <td> </td>
          <td>An object concept should no longer be used, but is not deleted because external data depend on it. This revision status applies, e.g. to definitions in shared descriptive terminologies.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="ExpertiseLevelEnum" id="ExpertiseLevelEnum">ExpertiseLevel</a>
    </h3>
    <p>Controlled vocabulary expressing the expertise exprected or required from human consumers of data or services. Values are restricted to integer values from 0 to 5. 0 is defined as unspecified level, and 1 to 5 indicates expertise from schoolchildren to taxonomic expert. See the description of the values for recommendations for interpreting and choosing the expert level.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>ExpertiseLevel0</td>
          <td>0 = Unspecified expertise level</td>
          <td> </td>
          <td>Use this if the expertise level of can not be assessed (e. g. when exporting data) or is considered irrelevant.</td>
        </tr>
        <tr class="data2">
          <td>ExpertiseLevel1</td>
          <td>1 = Elementary school (year 1 to 6)</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>ExpertiseLevel2</td>
          <td>2 = Middle school (year 7 to 10)</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>ExpertiseLevel3</td>
          <td>3 = High school (year 11 above) and general public</td>
          <td> </td>
          <td>When addressing this level specialized terminology or jargon should be avoided.</td>
        </tr>
        <tr class="data1">
          <td>ExpertiseLevel4</td>
          <td>4 = University students or (partly) trained staff</td>
          <td> </td>
          <td>This level uses specialized terminology, but avoids or explains problematic terms.</td>
        </tr>
        <tr class="data2">
          <td>ExpertiseLevel5</td>
          <td>5 = Experts</td>
          <td> </td>
          <td>This level uses the full range of terminology</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="ResourceTypeEnum" id="ResourceTypeEnum">ResourceType</a>
    </h3>
    <p>This enumeration is identical with the DCMI Type Vocabulary (http: //dublincore.org/documents/dcmi-terms/, as of 6/2004), except that an additional type "Other" has been added. Its purpose is to provide a framework of broad media or resource type terms, without the technical detail provided by the large number of MIME types. The annotations are largely based on those from the DublinCore metadata initiative vocabulary.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Collection</td>
          <td>Collection</td>
          <td> </td>
          <td>A collection is an aggregation of items. The term collection means that the resource is described as a group; its parts may be separately described and navigated.</td>
        </tr>
        <tr class="data2">
          <td>Dataset</td>
          <td>Dataset</td>
          <td> </td>
          <td>A dataset is information encoded in a defined structure (for example, lists, tables, and databases), intended to be useful for direct machine processing.</td>
        </tr>
        <tr class="data1">
          <td>Event</td>
          <td>Event</td>
          <td> </td>
          <td>An event is a non-persistent, time-based occurrence. Metadata for an event provides descriptive information that is the basis for discovery of the purpose, location, duration, responsible agents, and links to related events and resources. The resource of type event may not be retrievable if the described instantiation has expired or is yet to occur. Examples - exhibition, web-cast, conference, workshop, open-day, performance, battle, trial, wedding, tea-party, conflagration.</td>
        </tr>
        <tr class="data2">
          <td>Image</td>
          <td>Image</td>
          <td> </td>
          <td>An image is a primarily symbolic visual representation other than text. For example - images and photographs of physical objects, paintings, prints, drawings, other images and graphics, animations and moving pictures, film, diagrams, maps, musical notation. Note that image may include both electronic and physical representations.</td>
        </tr>
        <tr class="data1">
          <td>InteractiveResource</td>
          <td>Interactive Resource</td>
          <td> </td>
          <td>An interactive resource is a resource which requires interaction from the user to be understood, executed, or experienced. For example - forms on web pages, applets, multimedia learning objects, chat services, virtual reality.</td>
        </tr>
        <tr class="data2">
          <td>MovingImage</td>
          <td>Moving Image (Video)</td>
          <td> </td>
          <td>A series of visual representations that, when shown in succession, impart an impression of motion. Examples of moving images are: animations, movies, television programs, videos, zoetropes, or visual output from a simulation. Comment: Instances of the type "Moving Image" must also be describable as instances of the broader type "Image".</td>
        </tr>
        <tr class="data1">
          <td>PhysicalObject</td>
          <td>Physical Object</td>
          <td> </td>
          <td>An inanimate, three-dimensional object or substance. For example -- a computer, the great pyramid, a sculpture. Note that digital representations of, or surrogates for, these things should use Image, Text or one of the other types.</td>
        </tr>
        <tr class="data2">
          <td>Service</td>
          <td>Service</td>
          <td> </td>
          <td>A service is a system that provides one or more functions of value to the end-user. Examples include: a photocopying service, a banking service, an authentication service, interlibrary loans, a Z39.50 or Web server.</td>
        </tr>
        <tr class="data1">
          <td>Software</td>
          <td>Software</td>
          <td> </td>
          <td>Software is a computer program in source or compiled form which may be available for installation non-transiently on another machine. For software which exists only to create an interactive environment, use interactive instead.</td>
        </tr>
        <tr class="data2">
          <td>Sound</td>
          <td>Sound</td>
          <td> </td>
          <td>A sound is a resource whose content is primarily intended to be rendered as audio. For example - a music playback file format, an audio compact disc, and recorded speech or sounds.</td>
        </tr>
        <tr class="data1">
          <td>StillImage</td>
          <td>Still Image</td>
          <td> </td>
          <td>A static visual representation. Examples of still images are: paintings, drawings, graphic designs, plans and maps. Comment: Recommended best practice is to assign the type "Text" to images of textual materials. Instances of the type "Still Image" must also be describable as instances of the broader type "Image".</td>
        </tr>
        <tr class="data2">
          <td>Text</td>
          <td>Text</td>
          <td> </td>
          <td>A text is a resource whose content is primarily words for reading. For example - books, letters, dissertations, poems, newspapers, articles, archives of mailing lists. Note that facsimiles or images of texts are still of the genre text. Text may contain embedded still image illustrations, e. g., formatted html pages.</td>
        </tr>
        <tr class="data1">
          <td>Other</td>
          <td>Other</td>
          <td> </td>
          <td>Use this category if the resource does not seem to fit into any of the categories provided above.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TelephoneDeviceEnum" id="TelephoneDeviceEnum">TelephoneDevice</a>
    </h3>
    <p>Kind of phone number: voice, fax, mobile, pager, modem. These enumerated values are identical with vCard 3.0 flags (several of which can be added to a single phone number; to represent this in the UBIF interface duplicate the phone number itself!)</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Voice</td>
          <td>voice phone number</td>
          <td> </td>
        </tr>
        <tr class="data2">
          <td>Fax</td>
          <td>fax number</td>
          <td> </td>
        </tr>
        <tr class="data1">
          <td>Mobile</td>
          <td>mobile phone number</td>
          <td> </td>
        </tr>
        <tr class="data2">
          <td>Modem</td>
          <td>modem number</td>
          <td> </td>
        </tr>
        <tr class="data1">
          <td>Pager</td>
          <td>pager number</td>
          <td> </td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="Rating1to5Enum" id="Rating1to5Enum">Rating1to5</a>
    </h3>
    <p>Enumeration restricted to integer values from 1 to 5, indicating an arbitrary rating (meaning, e. g., 1 = disagree strongly, 2 = rather disagree, 3 = neutral or undecided, 4 = rather agree, 5 = agree strongly). This enumeration is of limited usefulness and could be replaced by an restriction on integer, but using the enumeration the semantics of agreement/disagreement or positive/negative rating can be communicated in a culture-neutral way (in German 1 is generally considered best and 5 worst, in English 1 worst, 5 best...).</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Rating0of5</td>
          <td>0</td>
          <td>Rating "0"</td>
          <td> </td>
          <td>Undecided (not yet rated).</td>
        </tr>
        <tr class="data2">
          <td>Rating1of5</td>
          <td>1</td>
          <td>Rating "1"</td>
          <td> </td>
          <td>For example, "disagree strongly", or "very poor".</td>
        </tr>
        <tr class="data1">
          <td>Rating2of5</td>
          <td>2</td>
          <td>Rating "2"</td>
          <td> </td>
          <td>For example, "rather disagree", or "poor".</td>
        </tr>
        <tr class="data2">
          <td>Rating3of5</td>
          <td>3</td>
          <td>Rating "3"</td>
          <td> </td>
          <td>For example, "neutral", "average", "undecided".</td>
        </tr>
        <tr class="data1">
          <td>Rating4of5</td>
          <td>4</td>
          <td>Rating "4"</td>
          <td> </td>
          <td>For example, "rather agree", or "good".</td>
        </tr>
        <tr class="data2">
          <td>Rating5of5</td>
          <td>5</td>
          <td>Rating "5"</td>
          <td> </td>
          <td>For example, "agree strongly", or "very good".</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TextDirectionalityEnum" id="TextDirectionalityEnum">TextDirectionality</a>
    </h3>
    <p>Values are ltr (left to right), rtl (right to left). Compare CSS2 and the XHTML 2.0 bi-directional text module. Note: A future UBIF version may also include lro/rlo = left-right-overide/right-left-overide, if this is found to be necessary.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>ltr</td>
          <td>ltr</td>
          <td>left-to-right text direction (e.g., English)</td>
          <td> </td>
        </tr>
        <tr class="data2">
          <td>rtl</td>
          <td>rtl</td>
          <td>right-to-left text direction (e.g., Arabic)</td>
          <td> </td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="StringFormattingTypeEnum" id="StringFormattingTypeEnum">StringFormattingType</a>
    </h3>
    <p>Controlled vocabulary expressing whether and which kind of inline formatting may be contained in a text literal (plain, inline-entity-encoded, html-level-entity-encoded). In the absence of an attribute providing further specification, most UBIF text elements may contain "inline" entity-escaped formatting! (Other standards: ~=atom:title/@type)</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>plain</td>
          <td>plain</td>
          <td>plain, unformatted text</td>
          <td> </td>
          <td>The element contains plain text with no entity escaped html. The entire text will contain the same format, i.e. it can contain not superscript, subscript or italicized parts. Example: "Mg2Cl &gt; 10mMol" (xml-entities like greater-than are still entity escaped!). Other standards: atom:title/@type=text.</td>
        </tr>
        <tr class="data2">
          <td>inline</td>
          <td>inline</td>
          <td>formatted text using selected html inline elements</td>
          <td> </td>
          <td>The element contains html formatting from an enumerated list of html inline formatting commands: strong, em, b, i, sub, sup, br. Other html commands (including inline formatting like a and img) are not recognized. Example: "Mg&lt;sub&gt;2&lt;/sub&gt;Cl &gt; 10mMol". Note: the constrained list of formats enables processors with limited implementation cost to render the formatting in other formats than the html-family.</td>
        </tr>
        <tr class="data1">
          <td>html</td>
          <td>html</td>
          <td>formatted text using html elements</td>
          <td> </td>
          <td>The element contains html formatting commands for an enumerated list of html inline formatting commands: strong, em, b, i, sub, sup, br. Other html commands (including inline formatting like a and img) are not recognized. Example: "Mg&lt;sub&gt;2&lt;/sub&gt;Cl &gt; 10mMol". Other standards: atom:title/@type=html. Note: atom also provides a value "xhtml", in which case the element content is directly full xhtml (i.e. contains mixed content). Such mixed content formatting was provided in early proposals of UBIF, but was rejected because it created significantly increased implementation cost in applications, including xslt in general and xml-database conversions. See EncodedFormattingProposal.zip for further information.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="MeasurementUnitPrefixEnum" id="MeasurementUnitPrefixEnum">MeasurementUnitPrefix</a>
    </h3>
    <p>Multiplication factor prefixes used in the scientific SI system (T, G, M, k, h, c, m, µ, n, p, f, a).</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Tera</td>
          <td>T</td>
          <td>tera</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 10 to the power of 12. Example: TW, terawatt.</td>
        </tr>
        <tr class="data2">
          <td>Giga</td>
          <td>G</td>
          <td>giga</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 10 to the power of 9. Example: GV, gigavolt.</td>
        </tr>
        <tr class="data1">
          <td>Mega</td>
          <td>M</td>
          <td>mega</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 10 to the power of 6. Example: MW, megawatt.</td>
        </tr>
        <tr class="data2">
          <td>kilo</td>
          <td>k</td>
          <td>kilo</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 1000. Example: kg, kilogram.</td>
        </tr>
        <tr class="data1">
          <td>hecto</td>
          <td>h</td>
          <td>hecto</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 100. Example: hPa, hectopascal.</td>
        </tr>
        <tr class="data2">
          <td>deca</td>
          <td>da</td>
          <td>deca</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 10. Example: daL, decaliter.</td>
        </tr>
        <tr class="data1">
          <td>noprefix</td>
          <td>-</td>
          <td>(none)</td>
          <td> </td>
          <td>No prefix, denoting factor or 1. Example: g, gram.</td>
        </tr>
        <tr class="data2">
          <td>deci</td>
          <td>d</td>
          <td>deci</td>
          <td> </td>
          <td>Prefix of measurement units, denoting a factor of 0.1. Example: dm, decimeter.</td>
        </tr>
        <tr class="data1">
          <td>centi</td>
          <td>c</td>
          <td>centi</td>
          <td> </td>
          <td>Prefix of measurement units, denoting a factor of 0.01. Example: cm, centimeter.</td>
        </tr>
        <tr class="data2">
          <td>milli</td>
          <td>m</td>
          <td>milli</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 10 to the power of -3. Example: mg, milligram.</td>
        </tr>
        <tr class="data1">
          <td>micro</td>
          <td>µ</td>
          <td>micro</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 10 to the power of -6. Example: µg, microgram.</td>
        </tr>
        <tr class="data2">
          <td>nano</td>
          <td>n</td>
          <td>nano</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 10 to the power of -9. Example: ng, nanogram.</td>
        </tr>
        <tr class="data1">
          <td>pico</td>
          <td>p</td>
          <td>pico</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 10 to the power of -12. Example: pmol, picomol.</td>
        </tr>
        <tr class="data2">
          <td>femto</td>
          <td>f</td>
          <td>femto</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 10 to the power of -15. Example: fm, femtometer.</td>
        </tr>
        <tr class="data1">
          <td>atto</td>
          <td>a</td>
          <td>atto</td>
          <td> </td>
          <td>Prefix of measurement units, denoting 10 to the power of -18. Example: attotesla.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="StatisticalMeasurementScaleEnum" id="StatisticalMeasurementScaleEnum">StatisticalMeasurementScale</a>
    </h3>
    <p>In statistical analysis it is often vital to know some basic properties of the values that are being analyzed. Some of these properties can be summarized in the form of a measurement scale. Higher scales can always be analyzed under the assumptions of a lower scale (ordinal data can be analyzed as nominal, ratio as interval).</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Nominal</td>
          <td>nominal</td>
          <td> </td>
          <td>unordered categorical states (DELTA character type 'UM')</td>
        </tr>
        <tr class="data2">
          <td>Ordinal</td>
          <td>ordinal</td>
          <td> </td>
          <td>ordered categorical states (DELTA character type 'OM'). Unless a separate tree defines more specific ordering, the order is assumed to be linear in the sequence in which the categories are enumerated in their definition.</td>
        </tr>
        <tr class="data1">
          <td>Interval</td>
          <td>interval</td>
          <td> </td>
          <td>real numeric (= floating point) values, where 0 is an arbitrarily defined point. As a consequence, ratios are undefined and only the intervals between values can be analyzed. Example: Temperature in °C or °F.</td>
        </tr>
        <tr class="data2">
          <td>Ratio</td>
          <td>ratio</td>
          <td> </td>
          <td>real numeric (= floating point) values (DELTA: type 'RN'), where 0 is an objective point and ratios can thus be analyzed. Example: length measurements. Most measures belong into this category and it is acceptable to assume the 'ratio' scale when importing DELTA legacy data.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="QuantitativeMeasurementScaleEnum" id="QuantitativeMeasurementScaleEnum">QuantitativeMeasurementScale</a>
    </h3>
    <p>Those values from StatisticalMeasurementScaleEnum addressing numerical data ('ratio' and 'interval').  
Note: Occasionally "integer" or "cardinal" (versus real numbers) are also considered part of the measurement scale. This should be avoided because: a) All combinations of interval/ratio and discrete/continous are possible. b) The important distinction is whether a measurement is based on a continuous or discrete scale. Although in most cases this is equivalent with integer versus real numbers, it is not necessarily so. An ANOVA will report false significance not only when values come from "1, 2, 3 and 4", but also when they come from "1.2, 2.4, 3.6 and 4.8".</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>Interval</td>
          <td> </td>
          <td>(see value "Interval" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>Ratio</td>
          <td> </td>
          <td>(see value "Ratio" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="CategoricalMeasurementScaleEnum" id="CategoricalMeasurementScaleEnum">CategoricalMeasurementScale</a>
    </h3>
    <p>Those values from StatisticalMeasurementScaleEnum addressing categorical data ('nominal' and 'ordinal').</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>Nominal</td>
          <td> </td>
          <td>(see value "Nominal" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>Ordinal</td>
          <td> </td>
          <td>(see value "Ordinal" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="DataStatusEnum" id="DataStatusEnum">DataStatus</a>
    </h3>
    <p>Status values (examples: "unknown", "not applicable") identify standardized reasons why data are missing. Alternative names are 'missing data indicators',  'special states', 'Null-values'. The annotation labels of the values can be freely changed as long as the semantics are preserved.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>ToBeChecked</td>
          <td>!</td>
          <td>To be checked</td>
          <td> </td>
          <td>Explicit indicator to revisit data later. This may occur when data are missing (known to exist, but not at hand for entering) or together with data (check  entered data against additional information source).</td>
        </tr>
        <tr class="data2">
          <td>ToBeIgnored</td>
          <td>ø</td>
          <td>Not to be recorded (a decision was made not enter data)</td>
          <td> </td>
          <td>(No DELTA equivalent exists. Implemented in Lucid 3 as "scope out".)</td>
        </tr>
        <tr class="data1">
          <td>NotApplicable</td>
          <td>-</td>
          <td>Not applicable (data are assumed to be impossible to exist)</td>
          <td> </td>
          <td>The character has been studied, but is not applicable. Example are situations where the character cannot possibly exists (e. g. leaf-tip in a leaf-less plant) or where the method is unable to measure the current situation (e. g. a growth rate if available space is used before the time defined in the method). In the first situation it may be desirable to create a character dependency definition, making the scoring of individual characters no longer necessary. (Note: the DELTA equivalent is the '-'-state.)</td>
        </tr>
        <tr class="data2">
          <td>DataUnavailable</td>
          <td>?</td>
          <td>Data unavailable (could not be obtained despite that an effort was made)</td>
          <td> </td>
          <td>The character has been studied and information has been searched, but not found. Use this to indicate that coding attempts should stop, perhaps until the next major revision. If data are currently unavailable, but attempts should continue, use '# = to be checked' instead. Compare also "not interpretable" if information found that is not interpretable. (Note: the DELTA equivalent is normally the 'U', but 'U' may also be used differently.)</td>
        </tr>
        <tr class="data1">
          <td>NotInterpretable</td>
          <td>#</td>
          <td>Not interpretable (data known to exist, but interpretation considered too uncertain).</td>
          <td> </td>
          <td>The character has been studied and information has been found, but is impossible to score because the information is not interpretable under the current terminology. Using this status warns collaborators that some information has already been studied (a specific annotation text accompanying the status value is recommend). Examples for using 'not interpretable': terms used in the source are undefined and cannot be guessed, the information in the source is contradictory, the color of a specimen is deteriorated so much that the original color can no longer be guessed. Note that in some cases an alternative to not coding at all may be to use certainty modifiers like 'perhaps' to make a tentative statement. (Note: no DELTA equivalent exists.)</td>
        </tr>
        <tr class="data2">
          <td>DataWithheld</td>
          <td>§</td>
          <td>Data withheld (exist in source of current document, but private or confidential).</td>
          <td> </td>
          <td> (Note: no DELTA equivalent exists.)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="UnivarAnyStatMeasureEnum" id="UnivarAnyStatMeasureEnum">UnivarAnyStatMeasure</a>
    </h3>
    <p>Enumerated list of univariate statistical methods. The list is intended to be more complete than normally necessary at least in biological morphometrics. If you still miss some measures, please request additions in a future version of this schema. Note: No satisfying external ontology for statistical methods could be found; the statistics section of MathML 2.0 (statistics.xsd) seems rather incomplete!</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>ObserverEstLower</td>
          <td>-</td>
          <td>Lower range limit (human estimate)</td>
          <td> </td>
          <td>Free estimate made by human observer for the lower range limit (no statistical sampling and calculation was used). This method is appropriate when it is known that the values are derived from experience with the described objects (perhaps from memory) or from scanning a sample of objects and measuring those objects considered 'typical'. This method is not appropriate for single measurements or for calculations based on statistical methods (which provide exact 'statistical estimates'). Compare also the 'UnknownMethod'-methods that are provided for legacy data.</td>
        </tr>
        <tr class="data2">
          <td>ObserverEstUpper</td>
          <td>+</td>
          <td>Upper range limit (human estimate)</td>
          <td> </td>
          <td>Free estimate made by human observer for the upper range limit (no statistical sampling and calculation was used). This method is appropriate when it is known that the values are derived from experience with the described objects (perhaps from memory) or from scanning a sample of objects and measuring those objects considered 'typical'. This method is not appropriate for single measurements or for calculations based on statistical methods (which provide exact 'statistical estimates'). Compare also the 'UnknownMethod'-methods that are provided for legacy data.</td>
        </tr>
        <tr class="data1">
          <td>ObserverEstCentral</td>
          <td>centr.</td>
          <td>Central or typical value (human estimate)</td>
          <td> </td>
          <td>Free estimate made by human observer for a single central or typical value (no statistical sampling and calculation was used). This method is appropriate when it is known that the values are derived from experience with the described objects (perhaps from memory) or from scanning a sample of objects and measuring those objects considered 'typical'. It is not appropriate for single measurements nor for calculations based on statistical methods (which provide exact 'statistical estimates'). Compare also the 'UnknownMethod'-methods that are provided for legacy data.</td>
        </tr>
        <tr class="data2">
          <td>UMethLower</td>
          <td>-(?)</td>
          <td>Lower range limit (legacy data, stat. meth. unknown)</td>
          <td> </td>
          <td>Lower range limit obtained by an unknown method (e. g. human observer estimate or some kind of statistical estimate). The range may, e. g., be mean plus/minus standard deviation, or a range estimate. 'Unknown' is important for legacy data where the statistical measure used is not known. If it is known that a measure is a human observer estimate rather than a defined value, the 'ObserverEstimate' methods should be used instead.</td>
        </tr>
        <tr class="data1">
          <td>UMethUpper</td>
          <td>+(?)</td>
          <td>Upper range limit (legacy data, stat. meth. unknown)</td>
          <td> </td>
          <td>Upper range limit obtained by an unknown method (e. g. human observer estimate or some kind of statistical estimate). The range may, e. g., be mean plus/minus standard deviation, or a range estimate. 'Unknown' is important for legacy data where the statistical measure used is not known. If it is known that a measure is a human observer estimate rather than a defined value, the 'ObserverEstimate' methods should be used instead.</td>
        </tr>
        <tr class="data2">
          <td>UMethCentral</td>
          <td>centr.(?)</td>
          <td>Central or typical value (legacy data, stat. meth. unknown)</td>
          <td> </td>
          <td>Central or typical value obtained by an unknown method (e. g. human observer estimate or some kind of statistical estimate). The central value may, e. g., be a single measurement, median, or arithmetic mean. 'Unknown' is important for legacy data where the statistical measure used is not known. If it is known that a measure is a human observer estimate rather than a defined value, the 'ObserverEstimate' methods should be used instead.</td>
        </tr>
        <tr class="data1">
          <td>Min</td>
          <td>Min</td>
          <td>Minimum value</td>
          <td> </td>
          <td>Absolute smallest value</td>
        </tr>
        <tr class="data2">
          <td>Max</td>
          <td>Max</td>
          <td>Maximum value</td>
          <td> </td>
          <td>Absolute largest value</td>
        </tr>
        <tr class="data1">
          <td>Mean</td>
          <td>µ</td>
          <td>Mean (= average)</td>
          <td> </td>
          <td>This is the normal, arithmetic mean.</td>
        </tr>
        <tr class="data2">
          <td>HMean</td>
          <td>hµ</td>
          <td>Harmonic mean</td>
          <td> </td>
          <td>The harmonic mean (reciprocal of the arithmetic mean of reciprocals) is rarely used. Recommendation: if nothing specific is said about a "mean", it can safely be assumed to be an arithmetic mean.</td>
        </tr>
        <tr class="data1">
          <td>GeoMean</td>
          <td>gµ</td>
          <td>Geometric mean</td>
          <td> </td>
          <td>The geometric mean (antilog of mean of logarithms) is relatively rarely used. Recommendation: if nothing specific is said about a "mean", it can safely be assumed to be an arithmetic mean.</td>
        </tr>
        <tr class="data2">
          <td>Mode</td>
          <td>mode</td>
          <td>Mode</td>
          <td> </td>
          <td>The value or value class with the highest frequency (most frequently occurring). Applicable only to unimodal distributions.</td>
        </tr>
        <tr class="data1">
          <td>Median</td>
          <td>med.</td>
          <td>Median</td>
          <td> </td>
          <td>The median is the 50 % percentile, i.e. 50% of the sampled values are smaller and the rest is larger than this value.</td>
        </tr>
        <tr class="data2">
          <td>InterqMean</td>
          <td>IQM</td>
          <td>Interquartile mean (= average)</td>
          <td> </td>
          <td>A truncated arithmetic mean, calculated only from those values that lie between 25 and 75% of sample values. This reduces the dependency of the mean on outliers and measurement errors.</td>
        </tr>
        <tr class="data1">
          <td>Var</td>
          <td>Var.</td>
          <td>Variance (sample, df = n-1)</td>
          <td> </td>
          <td>Variance based on a sample; calculated with n-1 (n = sample size) degrees of freedom. This is the "normal" variance used in almost all cases. A variance is a standard deviation squared.</td>
        </tr>
        <tr class="data2">
          <td>VarP</td>
          <td>Var. (pop.)</td>
          <td>Variance (population; df = n; rarely applicable!)</td>
          <td> </td>
          <td>Variance of population, calculated with n (= sample size) degrees of freedom. Use this if the entire population of objects has been studied. Normally conclusions about the population are based on a sample that has been studied; in this case the "normal" variance with df = n-1 is appropriate.</td>
        </tr>
        <tr class="data1">
          <td>SD</td>
          <td>s.d.</td>
          <td>Standard deviation (sample)</td>
          <td> </td>
          <td>Standard deviation based on a sample, calculated with n-1 (n = sample size) degrees of freedom. This is the "normal" standard deviation used in almost all cases.</td>
        </tr>
        <tr class="data2">
          <td>SDP</td>
          <td>s.d. (pop.)</td>
          <td>Standard deviation (population; df = n; rarely applicable!)</td>
          <td> </td>
          <td>Standard deviation based on the entire population; calculated with n (= sample size) degrees of freedom. Use this if the entire population of objects has been studied. Normally conclusions about the population are based on a sample that has been studied; in this case the "normal" std. dev. with df = n-1 is appropriate.</td>
        </tr>
        <tr class="data1">
          <td>MeanDev</td>
          <td>m.d.</td>
          <td>Mean deviation</td>
          <td> </td>
          <td>The mean of the absolute differences from the arithmetic mean of values. The absolute differences are the positive, unsquared differences from the mean.</td>
        </tr>
        <tr class="data2">
          <td>MeanDevMode</td>
          <td>m.d.m.</td>
          <td>Mean deviation from median</td>
          <td> </td>
          <td>The mean of the absolute differences from the median of values. The absolute differences are the positive, unsquared differences from the mode.</td>
        </tr>
        <tr class="data1">
          <td>CV</td>
          <td>CV</td>
          <td>Coefficient of variation (sample)</td>
          <td> </td>
          <td>Standard deviation (based on a sample), divided by the mean. The values entered should not be expressed as percent, but converted to a true value (use '0.3' for 30%). According to Sokal &amp; Rohlf 1981:59 this is a biased estimate, which may be corrected, compare 'CVC'.</td>
        </tr>
        <tr class="data2">
          <td>CVC</td>
          <td>CVC</td>
          <td>Corrected coefficient of variation (sample)</td>
          <td> </td>
          <td>Corrected coefficient corrected by (1 + (1/4n)). Compare, e. g., Sokal &amp; Rohlf 1981:59'.</td>
        </tr>
        <tr class="data1">
          <td>TotalRange</td>
          <td>TR</td>
          <td>Total range</td>
          <td> </td>
          <td>The maximum value minus the minimum value. Also often called "Range" without further qualification like 'absolute', 'total'. This measure can normally be computed automatically based on minimum and maximum. It will be manually entered, if minimum and maximum are not separately cited in a publication.</td>
        </tr>
        <tr class="data2">
          <td>InterqRange</td>
          <td>IQR</td>
          <td>Interquartile range</td>
          <td> </td>
          <td>This is the length of a symmetric interval around median, containing 50% of observations.</td>
        </tr>
        <tr class="data1">
          <td>SEMean</td>
          <td>s.e.</td>
          <td>Standard error of mean</td>
          <td> </td>
          <td>The standard error of mean is defined as: "std. dev. / square root(n)" (with n = sample size)</td>
        </tr>
        <tr class="data2">
          <td>SEVar</td>
          <td>s.e.(var.)</td>
          <td>Standard error of variance (of multiple samples)</td>
          <td> </td>
          <td>This is not a variance measure of the mean, but a measure of the variance of the variance estimates.</td>
        </tr>
        <tr class="data1">
          <td>Skew</td>
          <td>Skw.</td>
          <td>Skewness</td>
          <td> </td>
          <td>Coefficient of skewness of a distribution, a measure of the degree of asymmetry of a distribution around its mean: &lt;0 if mode &lt; median, =0 if symmetric, &gt; 0 if mode &gt; median.</td>
        </tr>
        <tr class="data2">
          <td>Kurt</td>
          <td>Kurt.</td>
          <td>Kurtosis</td>
          <td> </td>
          <td>Coefficient of kurtosis of distribution, a measure of the "peakedness" of a distribution.  A normal distribution has a value of 0.263, larger values indicate wider, smaller narrower distributions.</td>
        </tr>
        <tr class="data1">
          <td>N</td>
          <td>n</td>
          <td>Sample size</td>
          <td> </td>
          <td>The number of objects studied and on which the other reported statistical measures (mean, standard deviation, etc.) are based.</td>
        </tr>
        <tr class="data2">
          <td>ConfIntervalLower</td>
          <td>-CI{ParameterValue}</td>
          <td>Lower limit of {ParameterValue}% confidence interval for mean.</td>
          <td> </td>
          <td>The confidence interval is defined as a range into which the true mean of the distribution falls with a certain probability. The parameter expresses the confidence level in percent. Typical values are: 99.9% (= 0.05% from left!), 99% (= 0.5% from left), 95% (= 2.5% from left), 90% (= 5% from left).</td>
        </tr>
        <tr class="data1">
          <td>ConfIntervalUpper</td>
          <td>+CI{ParameterValue}</td>
          <td>Upper limit of {ParameterValue}% confidence interval for mean.</td>
          <td> </td>
          <td>The confidence interval is defined as a range into which the true mean of the distribution falls with a certain probability. The probability is expressed in percent in a parameter called; typical values are: 99.9 (= 99.95% from left!), 99 (= 99.5% from left), 95 (= 97.5% from left), 90 (= 95% from left).</td>
        </tr>
        <tr class="data2">
          <td>PercLower</td>
          <td>-P{ParameterValue}</td>
          <td>{ParameterValue}% percentile</td>
          <td> </td>
          <td>The {ParameterValue}% percentile is defined such that {ParameterValue}% of the observations are smaller than this value. Typical parameter values are 2.5, 5, 10, 15, 20, 25 (= 1st quartile), 30. Do not use 50; use the 'Median' measure instead!</td>
        </tr>
        <tr class="data1">
          <td>PercUpper</td>
          <td>+P{ParameterValue}</td>
          <td>{ParameterValue}% percentile</td>
          <td> </td>
          <td>The {ParameterValue}% percentile is defined such that {ParameterValue}% of the observations are smaller than this value. Typical parameter values are 70, 75 (= 3rd quartile), 80, 90, 95, 97.5. Do not use 50; use the 'Median' measure instead!</td>
        </tr>
        <tr class="data2">
          <td>TrimMean</td>
          <td>+TM{ParameterValue}</td>
          <td>{ParameterValue}% trim mean</td>
          <td> </td>
          <td>The arithmetic mean of the symmetric {ParameterValue}% interior portion of a set of data values.</td>
        </tr>
        <tr class="data1">
          <td>SDRangeLower</td>
          <td>µ - {ParameterValue} s.d.</td>
          <td>Mean minus {ParameterValue} stand. deviation(s)</td>
          <td> </td>
          <td>Lower limit of a range calculated as mean minus standard deviations. The parameter ParameterValue (here {ParameterValue}) defines a factor with which the s.d. is multiplied before it is substracted from the mean. Typical parameter values are 1 or 2.</td>
        </tr>
        <tr class="data2">
          <td>SDRangeUpper</td>
          <td>µ + {ParameterValue} s.d.</td>
          <td>Mean plus {ParameterValue} stand. deviation(s)</td>
          <td> </td>
          <td>Upper limit of a range calculated as mean plus standard deviations. The parameter ParameterValue (here {ParameterValue}) defines a factor with which the s.d. is multiplied before it is added to the mean. Typical parameter values are 1 or 2.</td>
        </tr>
        <tr class="data1">
          <td>MinOC</td>
          <td>Min\{ParameterValue} s.d.</td>
          <td>Minimum; outlier corrected ({ParameterValue} std. dev.)</td>
          <td> </td>
          <td>Absolute minimum value of sample, excluding outlier values more than {ParameterValue} standard deviations distant from the mean. Typical parameter values are 3 or 4.</td>
        </tr>
        <tr class="data2">
          <td>MaxOC</td>
          <td>Max\{ParameterValue} s.d.</td>
          <td>Maximum; outlier corrected ({ParameterValue} std. dev.)</td>
          <td> </td>
          <td>Absolute maximum value of sample, excluding outlier values more than {ParameterValue} standard deviations distant from the mean. Typical parameter values are 3 or 4.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="UnivarSimpleStatMeasureEnum" id="UnivarSimpleStatMeasureEnum">UnivarSimpleStatMeasure</a>
    </h3>
    <p>An enumeration of univariate statistical measures supported by UBIF (esp. used by SDD). Compare also UnivarParamStatMeasureEnum, containing further statistical measures that use an additional parameter (for percentage of percentile or confidence interval, etc.).</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>ObserverEstLower</td>
          <td> </td>
          <td>(see value "ObserverEstLower" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>ObserverEstUpper</td>
          <td> </td>
          <td>(see value "ObserverEstUpper" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>ObserverEstCentral</td>
          <td> </td>
          <td>(see value "ObserverEstCentral" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>UMethLower</td>
          <td> </td>
          <td>(see value "UMethLower" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>UMethUpper</td>
          <td> </td>
          <td>(see value "UMethUpper" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>UMethCentral</td>
          <td> </td>
          <td>(see value "UMethCentral" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>Min</td>
          <td> </td>
          <td>(see value "Min" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>Max</td>
          <td> </td>
          <td>(see value "Max" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>Mean</td>
          <td> </td>
          <td>(see value "Mean" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>HMean</td>
          <td> </td>
          <td>(see value "HMean" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>GeoMean</td>
          <td> </td>
          <td>(see value "GeoMean" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>Mode</td>
          <td> </td>
          <td>(see value "Mode" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>Median</td>
          <td> </td>
          <td>(see value "Median" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>InterqMean</td>
          <td> </td>
          <td>(see value "InterqMean" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>Var</td>
          <td> </td>
          <td>(see value "Var" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>VarP</td>
          <td> </td>
          <td>(see value "VarP" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>SD</td>
          <td> </td>
          <td>(see value "SD" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>SDP</td>
          <td> </td>
          <td>(see value "SDP" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>MeanDev</td>
          <td> </td>
          <td>(see value "MeanDev" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>MeanDevMode</td>
          <td> </td>
          <td>(see value "MeanDevMode" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>CV</td>
          <td> </td>
          <td>(see value "CV" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>CVC</td>
          <td> </td>
          <td>(see value "CVC" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>TotalRange</td>
          <td> </td>
          <td>(see value "TotalRange" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>InterqRange</td>
          <td> </td>
          <td>(see value "InterqRange" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>SEMean</td>
          <td> </td>
          <td>(see value "SEMean" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>SEVar</td>
          <td> </td>
          <td>(see value "SEVar" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>Skew</td>
          <td> </td>
          <td>(see value "Skew" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>Kurt</td>
          <td> </td>
          <td>(see value "Kurt" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>N</td>
          <td> </td>
          <td>(see value "N" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="UnivarParamStatMeasureEnum" id="UnivarParamStatMeasureEnum">UnivarParamStatMeasure</a>
    </h3>
    <p>An enumeration of parameterized univariate statistical measures supported by UBIF (esp. used by SDD). This enumeration is similar to UnivarSimpleStatMeasureEnum, but here a parameter value is supported. Abbreviation, Label and Details (within xs:documentation) may contain the string "{ParameterValue}". Labels are expected to become meaningful if this is replaced with the actual parameter value.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>ConfIntervalLower</td>
          <td> </td>
          <td>(see value "ConfIntervalLower" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>ConfIntervalUpper</td>
          <td> </td>
          <td>(see value "ConfIntervalUpper" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>PercLower</td>
          <td> </td>
          <td>(see value "PercLower" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>PercUpper</td>
          <td> </td>
          <td>(see value "PercUpper" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>TrimMean</td>
          <td> </td>
          <td>(see value "TrimMean" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>SDRangeLower</td>
          <td> </td>
          <td>(see value "SDRangeLower" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>SDRangeUpper</td>
          <td> </td>
          <td>(see value "SDRangeUpper" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>MinOC</td>
          <td> </td>
          <td>(see value "MinOC" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>MaxOC</td>
          <td> </td>
          <td>(see value "MaxOC" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="AgentRoleEnum" id="AgentRoleEnum">AgentRole</a>
    </h3>
    <p>Provides codes for roles like author, editor, photographer, advisor, or copyright holder. The roles and their codes used here are based on http://www.loc.gov/marc.relators/ (as of 2004/6 available at http://dublincore.org/usage/meetings/2004/03/Relator-codes.html). For example, the enumerated code "aut" for author corresponds to http://www.loc.gov/marc.relators/aut. The DublinCore Agents group is considering using the same codes (see e. g. http://www.loc.gov/marc/dc/Agent-roles.html), but as of 2004/6 the DublinCore Agents subgroup did not yet agree on a Creator/Contributor refinement as qualified DublinCore. Note that the roles selected here are a subset of the roles that are available in MARC. See second annotation for reasons of not using a union-design, which would be easier to maintain!</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>aut</td>
          <td>Author</td>
          <td> </td>
          <td>A person or corporate body chiefly responsible for the intellectual or artistic content of a work. This term may also be used when more than one person or body bears such responsibility.</td>
        </tr>
        <tr class="data2">
          <td>edt</td>
          <td>Editor</td>
          <td> </td>
          <td>A person who prepares for publication a work not primarily his/her own, such as by elucidating text, adding introductory or other critical matter, or technically directing an editorial staff.</td>
        </tr>
        <tr class="data1">
          <td>cre</td>
          <td>Creator in general</td>
          <td> </td>
          <td>A person or corporate body responsible for the intellectual or artistic content of a work. The more specific type Author [aut] or Editor [edt] should be preferred.</td>
        </tr>
        <tr class="data2">
          <td>ill</td>
          <td>Illustrator</td>
          <td> </td>
          <td>The person who conceives, and perhaps also implements, a design or illustration, usually to accompany a written text.</td>
        </tr>
        <tr class="data1">
          <td>pht</td>
          <td>Photographer</td>
          <td> </td>
          <td>The person or organization responsible for taking photographs, whether they are used in their original form or as reproductions.</td>
        </tr>
        <tr class="data2">
          <td>ctb</td>
          <td>Contributor in general</td>
          <td> </td>
          <td>Someone whose work has been contributed to a larger work, such as an anthology, serial publication, or other compilation of individual works. Do not use for someone whose sole function in relation to a work is as author, editor, compiler or translator.</td>
        </tr>
        <tr class="data1">
          <td>trl</td>
          <td>Translator</td>
          <td> </td>
          <td>A person who renders a text from one language into another, from an older form of a language into the modern form, or from one audience-specific representation to one appropriate for another audience.</td>
        </tr>
        <tr class="data2">
          <td>trc</td>
          <td>Transcriber</td>
          <td> </td>
          <td>A person who prepares a handwritten or typewritten copy from original material, including from dictated or orally recorded material.</td>
        </tr>
        <tr class="data1">
          <td>clb</td>
          <td>Collaborator</td>
          <td> </td>
          <td>A person or corporate body that takes a limited part in the elaboration of a work of another person or corporate body that brings complements (e.g., appendices, notes) to the work.</td>
        </tr>
        <tr class="data2">
          <td>col</td>
          <td>Collector</td>
          <td> </td>
          <td>A person who has brought together material from various sources, which has been arranged, described, and catalogued as a collection. The collector is neither the creator of the material nor the person to whom manuscripts in the collection may have been addressed.</td>
        </tr>
        <tr class="data1">
          <td>crp</td>
          <td>Correspondent</td>
          <td> </td>
          <td>A person or organization who was either the writer or recipient of a letter or other communication.</td>
        </tr>
        <tr class="data2">
          <td>prg</td>
          <td>Programmer</td>
          <td> </td>
          <td>A person or corporate body responsible for the creation and/or maintenance of computer program design documents, source code, and machine-executable digital files and supporting documentation.</td>
        </tr>
        <tr class="data1">
          <td>rth</td>
          <td>Research team head</td>
          <td> </td>
          <td>The person or corporate body that directed or managed a research project.</td>
        </tr>
        <tr class="data2">
          <td>rtm</td>
          <td>Research team member</td>
          <td> </td>
          <td>The person or corporate body that participated in a research project but whose role did not involve direction or management of it.</td>
        </tr>
        <tr class="data1">
          <td>res</td>
          <td>Researcher</td>
          <td> </td>
          <td>The person or corporate body responsible for performing research.
</td>
        </tr>
        <tr class="data2">
          <td>sad</td>
          <td>Scientific advisor</td>
          <td> </td>
          <td>A person who brings scientific, pedagogical, or historical competence to the conception and realization on a work, particularly in the case of audio-visual items.
</td>
        </tr>
        <tr class="data1">
          <td>pfr</td>
          <td>Proofreader</td>
          <td> </td>
          <td>A person who corrects text (orthography, grammar).</td>
        </tr>
        <tr class="data2">
          <td>mrk</td>
          <td>Markup editor</td>
          <td> </td>
          <td>The person or organization performing the coding of SGML, HTML, or XML markup of metadata, text, etc.</td>
        </tr>
        <tr class="data1">
          <td>cmm</td>
          <td>Commentator</td>
          <td> </td>
          <td>A person who provides interpretation, analysis, or a discussion of the subject matter on a recording, motion picture, or other audiovisual medium.</td>
        </tr>
        <tr class="data2">
          <td>rev</td>
          <td>Reviewer</td>
          <td> </td>
          <td>A person or corporate body responsible for the review of book, motion picture, performance, etc.</td>
        </tr>
        <tr class="data1">
          <td>csl</td>
          <td>Consultant</td>
          <td> </td>
          <td>The person called upon for professional advice or services in a specialized field of knowledge or training.</td>
        </tr>
        <tr class="data2">
          <td>own</td>
          <td>Owner</td>
          <td> </td>
          <td>The person or organization that currently owns an item or collection.</td>
        </tr>
        <tr class="data1">
          <td>fmo</td>
          <td>Former owner</td>
          <td> </td>
          <td>The person or organization who owned an item at any time in the past. Includes those to whom the material was once presented. The person or organization giving the item to the present owner is designated as Donor [dnr]</td>
        </tr>
        <tr class="data2">
          <td>cph</td>
          <td>Copyright holder</td>
          <td> </td>
          <td>A person or organization owning the copyright of the material.</td>
        </tr>
        <tr class="data1">
          <td>cpc</td>
          <td>Copyright claimant</td>
          <td> </td>
          <td>The person listed as a copyright owner at the time of registration. Copyright can be granted or later transferred to another person or agent, at which time the claimant becomes the copyright holder.</td>
        </tr>
        <tr class="data2">
          <td>dnr</td>
          <td>Donor</td>
          <td> </td>
          <td>The donor of a book, manuscript, etc., to its present owner. Donors to previous owners are designated as Former owner [fmo]</td>
        </tr>
        <tr class="data1">
          <td>dpt</td>
          <td>Depositor</td>
          <td> </td>
          <td>A person or organization placing material in the physical custody of a library or repository without transferring the legal title.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="AgentCreatorContribRoleEnum" id="AgentCreatorContribRoleEnum">AgentCreatorContribRole</a>
    </h3>
    <p>Union of AgentCreatorRoleEnum and AgentContributorRoleEnum, but no Owner roles. Technical note: currently this is modeled somewhat strangely as an xml-schema restriction of AgentRoleEnum (= union of all three basic role groups). This is a workaround for a problem Xerxes 2.6.2 detected by Jacob Asiedu! We hope that in the future the whole 'xs:restriction' below can be replaced again with the much more straightforward 'xs:union memberTypes="AgentCreatorRoleEnum AgentContributorRoleEnum"/' (= union of the two intended AgentRole enumerations).</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>aut</td>
          <td> </td>
          <td>(see value "aut" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>edt</td>
          <td> </td>
          <td>(see value "edt" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>cre</td>
          <td> </td>
          <td>(see value "cre" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>ill</td>
          <td> </td>
          <td>(see value "ill" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>pht</td>
          <td> </td>
          <td>(see value "pht" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>ctb</td>
          <td> </td>
          <td>(see value "ctb" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>trl</td>
          <td> </td>
          <td>(see value "trl" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>trc</td>
          <td> </td>
          <td>(see value "trc" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>clb</td>
          <td> </td>
          <td>(see value "clb" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>col</td>
          <td> </td>
          <td>(see value "col" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>crp</td>
          <td> </td>
          <td>(see value "crp" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>prg</td>
          <td> </td>
          <td>(see value "prg" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>rth</td>
          <td> </td>
          <td>(see value "rth" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>rtm</td>
          <td> </td>
          <td>(see value "rtm" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>res</td>
          <td> </td>
          <td>(see value "res" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>sad</td>
          <td> </td>
          <td>(see value "sad" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>pfr</td>
          <td> </td>
          <td>(see value "pfr" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>mrk</td>
          <td> </td>
          <td>(see value "mrk" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>cmm</td>
          <td> </td>
          <td>(see value "cmm" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>rev</td>
          <td> </td>
          <td>(see value "rev" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>csl</td>
          <td> </td>
          <td>(see value "csl" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="AgentCreatorRoleEnum" id="AgentCreatorRoleEnum">AgentCreatorRole</a>
    </h3>
    <p>Enumeration of roles supported for creator agents. See AgentRoleEnum for information about the MARC relator codes.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>aut</td>
          <td> </td>
          <td>(see value "aut" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>edt</td>
          <td> </td>
          <td>(see value "edt" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>cre</td>
          <td> </td>
          <td>(see value "cre" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>ill</td>
          <td> </td>
          <td>(see value "ill" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>pht</td>
          <td> </td>
          <td>(see value "pht" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="AgentContributorRoleEnum" id="AgentContributorRoleEnum">AgentContributorRole</a>
    </h3>
    <p>Enumeration of supported roles for contributor agents. See AgentRoleEnum for information about the MARC relator codes.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>ctb</td>
          <td> </td>
          <td>(see value "ctb" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>trl</td>
          <td> </td>
          <td>(see value "trl" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>trc</td>
          <td> </td>
          <td>(see value "trc" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>clb</td>
          <td> </td>
          <td>(see value "clb" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>col</td>
          <td> </td>
          <td>(see value "col" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>crp</td>
          <td> </td>
          <td>(see value "crp" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>prg</td>
          <td> </td>
          <td>(see value "prg" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>rth</td>
          <td> </td>
          <td>(see value "rth" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>rtm</td>
          <td> </td>
          <td>(see value "rtm" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>res</td>
          <td> </td>
          <td>(see value "res" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>sad</td>
          <td> </td>
          <td>(see value "sad" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>pfr</td>
          <td> </td>
          <td>(see value "pfr" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>mrk</td>
          <td> </td>
          <td>(see value "mrk" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>cmm</td>
          <td> </td>
          <td>(see value "cmm" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>rev</td>
          <td> </td>
          <td>(see value "rev" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>csl</td>
          <td> </td>
          <td>(see value "csl" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="AgentOwnerRoleEnum" id="AgentOwnerRoleEnum">AgentOwnerRole</a>
    </h3>
    <p>Enumeration of supported roles for owner/copyright agents. See AgentRoleEnum for information about the MARC relator codes.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>own</td>
          <td> </td>
          <td>(see value "own" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>fmo</td>
          <td> </td>
          <td>(see value "fmo" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>cph</td>
          <td> </td>
          <td>(see value "cph" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>cpc</td>
          <td> </td>
          <td>(see value "cpc" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>dnr</td>
          <td> </td>
          <td>(see value "dnr" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>dpt</td>
          <td> </td>
          <td>(see value "dpt" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="LabelRoleEnum" id="LabelRoleEnum">LabelRole</a>
    </h3>
    <p>Controlled vocabulary expressing the kind of label text. These are currently highly constrained, but either additional values or free extensibility (by union of this enum type with xs:anyURI) are expected for future releases of UBIF.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Full</td>
          <td>full</td>
          <td>full label</td>
          <td> </td>
          <td>Free-form label text designed to be as concise and expressive as possible while remaining intuitive and (locally) unique. Depending on the object type, this label may often be called 'name' or a 'title'. The recommended length is less than 80 characters; longer text (up to 255 char.) is supported but human user interface software may be optimized for shorter labels. Text is recommended to be unique within dataset, object type, and language/audience combination. A full label may contain abbreviations but is not usually considered an abbreviated label itself. It should uniquely identify an object, but multiple full labels may exist. -- Examples: Agent 'Gregor Hagedorn, BBA, Berlin', 'G. Hagedorn, Biol. Bundesanstalt'; Taxon 'Ascochyta aceris Fuckel', 'Ascochyta aceris Lib.', 'Ascochyta aceris Sacc.' or 'Ascochyta aceris Saccardo'. -- Other standards: ~= DC.Title.</td>
        </tr>
        <tr class="data2">
          <td>Short</td>
          <td>short</td>
          <td>short label</td>
          <td> </td>
          <td>Short label text designed to represent an object in user interfaces or reports (esp. in tabular format) where space considerations are more important than intuitive understanding or readability. The text short label may be context dependent and may contain unusual abbreviations that require knowledge of the full/concise label to be intelligable. The recommended length is less than 20 characters (including blanks); longer text (up to 255 char.) is supported but may not be readable in all human user interface designs. When missing, applications may attempt to abbreviate the full label, which may result in duplicate labels, however.  Text is recommended to be unique within dataset, object type, and language/audience combination. -- Examples: Agent 'G. Hagedorn'; Taxon 'Asc. aceris Fck.'.</td>
        </tr>
        <tr class="data1">
          <td>Abbrev</td>
          <td>abbrev.</td>
          <td>abbreviation</td>
          <td> </td>
          <td>A label text abbreviated as far as possible while remaining unique within a dataset. When missing, applications may attempt to abbreviate the short or full label, which may result in duplicate labels, however.  Text is recommended to be unique within dataset, object type, and language/audience combination. -- Examples: Agent 'GH'; Taxon 'A.aceris' (assuming the dataset contains no other Genus abbreviated as 'A.' with a species epithet 'aceris').</td>
        </tr>
        <tr class="data2">
          <td>Token</td>
          <td>token</td>
          <td>token/variable name</td>
          <td> </td>
          <td>A highly constrained version of the label designed for exports to applications requiring very short and simple object names, such as fields in databases and variable names in phylogenetic or statistical analysis software like NEXUS or SAS, which often must be short, not having blanks or punctuation, etc. The application for which a token label is designed may be specified using a custom language value, e.g. 'x-SAS', 'x-NEXUS'. -- Examples: Agent 'GHAGE'; Taxon 'AscAceris'.</td>
        </tr>
        <tr class="data1">
          <td>Sort</td>
          <td>sort.</td>
          <td>sorting string</td>
          <td> </td>
          <td>A string that results in a desired sorting order when sorted with a standard, case-insensitive unicode sorting routine. The most important problems addressed here are word or token order, leading articles, and quoting marks (see examples). Other problems (special locale-specific sorting rules for accented characters like 'ü'='ue' in German but 'u' in Spanish, 'rr/ll' sorted after 'r/l' in Spanish, or character ligatures may or may not be addressed, depending on the data available to the provider. Where the provider has a choice, it is recommended *not* to address these problems, since sorting may be desirable according to rules defines defined in the locale of the consumer, not the provider. Current unicode-based data handling software often provides sufficient locale-specific sorting rules for accented or ligature characters. Problem: The provider may have stored all-uppercase sort-strings. This is not recommened, but consumers should expect that the sorting string has this form. One consequence is that the sorting string, while the best available choice for sorting where provided, may not be suitable for labeling the data in a sorted list. Should a second label type ('sortreport'?) be defined for this purpose? -- Examples: Agent 'Duarte, Amália Mourinha' (pt), 'Pina de Morales, Ana Maria' (es).; Taxon 'Isoetes' for if preferred form uses 'Isoëtes' (as allowable under ICBN); MediaObject 'high plano, the' for image titled 'The "high" plano'. -- Other applicable standards: vCard:Sort-String</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="DetailRoleEnum" id="DetailRoleEnum">DetailRole</a>
    </h3>
    <p>Controlled vocabulary expressing the kind of long 'detail' text. In contrast to label text, detail text to which these values are applied are not restricted in length. Note that the text length of detail text elements is not limited by the schema. It is, however, recommended that the length does not exceed 30000 characters because longer text may lead to interoperability problems. Currently the enumerated values are highly constrained, but either extension or free extensibility (by union of this enum type with xs:anyURI) is expected for future releases of UBIF. Note that a 'description' is usually presented instead of the object, and a 'caption' always together with the object. An 'abstract' may be presented instead or together with the object.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Description</td>
          <td>descript.</td>
          <td>Description</td>
          <td> </td>
          <td>Free-form text describing an object. It may summarize the information contained in the object (similar to an abstract) but may also inform about related information. -- Other standards: ~= DC.Description.</td>
        </tr>
        <tr class="data2">
          <td>Caption</td>
          <td>caption</td>
          <td>Caption</td>
          <td> </td>
          <td>A special form of free-form description that is expected to be displayed together with the object (especially images, video, or audio formats). Caption as defined here is limited to a caption displayed together with the entire object (i. e. excluding timed captions/subtitles of audio/video objects). Frequently description and caption texts are identical, in which case providing the description is sufficient. Specific captions may contain, e. g., references to places or objects in the image such as 'in upper left corner ...', 'bar = 10µm'.</td>
        </tr>
        <tr class="data1">
          <td>Abstract</td>
          <td>abstr.</td>
          <td>Abstract</td>
          <td> </td>
          <td>A special form of free-form description that is contains a condensed version of the object content itself. Detail text with role 'abstract' is commonly used in the case of publications. The main difference to a 'description' is that an abstract does not contain external or metainformation about the object. Its applicability is therefore more limited, but also the correspondence of the representation with the full object may be assumed to be larger.</td>
        </tr>
        <tr class="data2">
          <td>Coverage</td>
          <td>cover.</td>
          <td>Coverage</td>
          <td> </td>
          <td>Free-form text describing geographic, taxonomic, or other coverage aspects of the object or data set. -- Other standards: ~= DC.Coverage.</td>
        </tr>
        <tr class="data1">
          <td>NormativeText</td>
          <td>norm.</td>
          <td>Normative definition</td>
          <td> </td>
          <td>Free-form text defining the concept of the object (applicable only to conceptual objects, not actual objects like specimens or books). Similar to description, but considered normative or translated-normative. Desirable external validation: only a single normative text should be without a "translatedfrom" attribute, all other should specify a translation source (and are "translated-normative", not "fully normative").</td>
        </tr>
        <tr class="data2">
          <td>UnknownDetailRole</td>
          <td>?</td>
          <td>Unknown</td>
          <td> </td>
          <td>The role ('Description', 'Caption', 'Abstract') of long detail text is unknown.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="MediaRepresentationRoleEnum" id="MediaRepresentationRoleEnum">MediaRepresentationRole</a>
    </h3>
    <p>Controlled vocabulary expressing the object representation role a media object (image, audio/video, rich text) may have. </p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Iconic</td>
          <td>iconic</td>
          <td>icon/thumbnail (presented inline, if possible)</td>
          <td> </td>
          <td>Small image or short audio/video resource to supplement, but usually not replace label text. Most media resources of this role will not be informative enough to represent an object without additional textual representation. Recommendation: Typically only a single icon image should be defined for each object; applications may not be able to display more than one. However, combinations of still image and audio resource should be expected by applications supporting audio resources.</td>
        </tr>
        <tr class="data2">
          <td>Primary</td>
          <td>primary</td>
          <td>primary representation (presented inline, if possible)</td>
          <td> </td>
          <td>Media object intended as a first class representation of the object, fully informative without additional textual representation. If multiple media resources of this role are defined for an object, all of these are expected to be available when making the selection decision.</td>
        </tr>
        <tr class="data1">
          <td>Diagnostic</td>
          <td>diagn.</td>
          <td>representation optimized for selecting one out of several alternatives (presented inline, if possible)</td>
          <td> </td>
          <td>A subclass of primary media object representations, optimized for being displayed in situations where a human user is expected to choose (select) among multiple instances. Typically the size, duration, or complexity of these resources will be less than in a primary resource. Examples: in a multi-access picture key, the human selects one or several states describing characteristics of a specimen to be identified, or selected one of several images representing the remaining taxa. </td>
        </tr>
        <tr class="data2">
          <td>Secondary</td>
          <td>2ndary</td>
          <td>additional, secondary, or supplementary representations (available on request)</td>
          <td> </td>
          <td>Media objects intended to inform the user rather on request. For example, a user interface might provide a single link/button to request additional information, or a short list of links allowing direct access to individual media objects. Examples are images of a character state in a specific taxon, which illustrates the concept, but does not expresses its circumscription.</td>
        </tr>
        <tr class="data1">
          <td>Normative</td>
          <td>normat.</td>
          <td>normative resource for a concept definition</td>
          <td> </td>
          <td>Detailed information used to define a resource object. Examples are images or rich formatted text (glossary entry, a wiki page, or a taxon/species page). Note: abstract concepts (taxon, character, publication) or physical objects (such as specimens) itself may be identified by URI in Link rel="BasedOn". Reflexive usage: recording that The URI may be identical to a definition resource identified through this resource role (but is not required to be).</td>
        </tr>
        <tr class="data2">
          <td>UnknownMediaRole</td>
          <td>?</td>
          <td>unknown</td>
          <td> </td>
          <td>The role of the media object is unknown (explictly equivalent to not specifying a value for the role).</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="LinkingRelEnum" id="LinkingRelEnum">LinkingRel</a>
    </h3>
    <p>Controlled vocabulary expressing the semantics of a Link uri. The vocabulary includes all values from IRPStatementRoleEnum. These are currently highly constrained, but future version may add either more values or free extensibility (union of this type with xs:anyURI).</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>LatestVersion</td>
          <td>latest</td>
          <td>latest version</td>
          <td> </td>
          <td>Link resolving to a representation of the latest (i.e. newest) version of the object at the time of resolution. This explicitly assumes that the result of resolution changes over time (non-persistent resource).</td>
        </tr>
        <tr class="data2">
          <td>PriorVersion</td>
          <td>prev.</td>
          <td>prev. version</td>
          <td> </td>
          <td>Link resolving to the previous version of the current object. [Not related to xhtml prev]</td>
        </tr>
        <tr class="data1">
          <td>Alternate</td>
          <td>alt.</td>
          <td>alternate</td>
          <td> </td>
          <td>Link designating an alternate (substitute) representation for the object in which this link occurs. These representations may differ in language, format, level of detail, etc. All alternate representations are implicitly about the same abstract resource, and it is recommended to express the abstract resource using rel="about". [Closely related to xhtml 1.0 "alternate" recommended link type value.]</td>
        </tr>
        <tr class="data2">
          <td>SubclassOf</td>
          <td>Subclass</td>
          <td>Subclass Of</td>
          <td> </td>
          <td>Generalization relationship pointing to the more general concept (i.e. this concept is a kind-of linked concept).
Example: 'teliospore' and 'conidium are both a kind of 'spore', a 'spore' is a kind of 'propagule'). In the context of phylogenetic analyses a distinction between "homologous" and "heterologous generalizations" may be necessary, this is considered out of the general scope here and may be implemented in external ontologies.</td>
        </tr>
        <tr class="data1">
          <td>SuperclassOf</td>
          <td>Superclass</td>
          <td>Superclass Of</td>
          <td> </td>
          <td>Inverse of subclass of; generalization relationship pointing to the less general class.</td>
        </tr>
        <tr class="data2">
          <td>BasedOn</td>
          <td>BasedOn</td>
          <td>based on</td>
          <td> </td>
          <td>The current resource is based on information from another resource, in a way that is different from verbatim copying and different from the extension semantics associated with sub/superclass relations. Specifically, this includes concepts like "source", "origin", or "derivation". "BasedOn" is transitive, not symmetric, not reflexive. Persistence of the other resource is assumed. The current resource may have a similar or entirely different format than the one it is based on. Examples: 1) A record based on a physical specimen (which has a urn). 2) A reference record based on a publication that has a DOI identifier. 3) A character definition based on a standard character definition.</td>
        </tr>
        <tr class="data1">
          <td>SameAs</td>
          <td>SameAs</td>
          <td>same as</td>
          <td> </td>
          <td>Two resources have same semantics, same creator, same presentation format, but are not necessarily bitwise identical (e. g., sequence of elements in a set changed). SameAs is transitive, symmetric, reflexive. Persistence of the other resource is assumed. Note that multiple URIs of the current object may be given in "current", but this designates these as equal alternatives. Citing only one URI in "current" and others in "SameAs" may inform about older, outdated URIs.</td>
        </tr>
        <tr class="data2">
          <td>SimilarTo</td>
          <td>SimilarTo</td>
          <td>similar to</td>
          <td> </td>
          <td>Both resources (indepently created or of unknown derivation) are analyzed to be similar. "SimilarTo" is symmetric, reflexive, not transitive (problem of chaining). Whether a data consumer desires to make inferences on the weak "SimilarTo" relation or not is its own decision. In the case of character-based identifications, the built-in error tolerance mechansisms of the identification method will often allow doing so. Persistence of the other resource is assumed. Examples: 1) A character state may be (more or less roughly) equivalent to another in another dataset, although neither definition is based on the other. 2) Multiple images may be taken from the same object.</td>
        </tr>
        <tr class="data1">
          <td>IsDifferentFrom</td>
          <td>Diff. from</td>
          <td>Is 
different from</td>
          <td> </td>
          <td>Opposite assertion to "SimilarTo". This asserts that two resources may appear related, but are not. "IsDifferentFrom" is symmetric, not transitive, not reflexive. Persistence of the other resource is assumed.</td>
        </tr>
        <tr class="data2">
          <td>PartOf</td>
          <td>Part of</td>
          <td>Part of</td>
          <td> </td>
          <td>Indicating composition. Data may be part of other data (a), a physical object may be part of another physical object (b), or an abstract concept may be a part of another concept (c). Examples: plant = root/stem/leaf, leaf = base/stipules/petiole/lamina, etc.; medium preparation and inoculation methods are part of microbial growth rate measurement method.
PartOf is transitive, anti-symmetric, and reflexive (though is of theoretical use only). Detailed information on use case (a): A dataset may be a subset of a larger dataset; if a dataset is dynamically generated in response to a query it may either have a URI encoding the query or no URI at all (i. e. uri-attribute on object itself does not exist), but may want to publish the URI or its base-dataset; one purpose of this is that a publisher may want to inform indexers (crawlers) about a resolvable URI to the larger dataset, another to establish the status of archival data. - On use case b): A microscopic slide may be part of a specimen in a collection. On use case c): A method may be a part (step) of a composite method, an article part of a publication, a petal is a part of the plant flower).</td>
        </tr>
        <tr class="data1">
          <td>HasPart</td>
          <td>has part</td>
          <td>has part</td>
          <td> </td>
          <td>Inverse of the part-of relation, indicating the parts that something consists of. A book may have resources for chapters that are independently retrievable, a specimen unit may have subunits, a descriptive term like "ovate to elliptic" has the parts "ovate" and "elliptic" (the kind of composition being not defined here!).</td>
        </tr>
        <tr class="data2">
          <td>BackwardCompatibleWith</td>
          <td>Backw. Compat.</td>
          <td>Backward Compatible With</td>
          <td> </td>
          <td>The linked resource (usually a prior version) is backward compatible with the current resource. Examples: A new species has been added to a genus. The old enumerated taxon concept is backwards compatible with the new one. Related to: owl:backwardCompatibleWith.</td>
        </tr>
        <tr class="data1">
          <td>IncompatibleWith</td>
          <td>Incompat.</td>
          <td>Incompatible With</td>
          <td> </td>
          <td>The linked resource (usually a prior version) is incompatible compatible with the current resource. Examples: A categorical character with 3 states (which partion a continous value space) now has 4 states. The addition of the forth state not only extends the value space (as in adding "ultra-violet", which might be BackwardCompatibleWith), but changes the borders of the other states (as in adding "orange" between "red" and "yellow". Related to: owl:incompatibleWith.</td>
        </tr>
        <tr class="data2">
          <td>Meta</td>
          <td>meta</td>
          <td>meta</td>
          <td> </td>
          <td>Link referring to a resource offering meta data or help (more information about the current resource, including meta data like copyright or licensing, links to other sources information, etc.). Compare IRPStatements with role="Copyright", "License", "Acknowledgement", etc. to express more specific relations. [Related to xhtml 2.0 "meta", but also to xhtml 1.0-2.0 "help" recommended link type values.]</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="IRPStatementRoleEnum" id="IRPStatementRoleEnum">IRPStatementRole</a>
    </h3>
    <p>Controlled vocabulary expressing the kind of IPR (= intellectual property right) statements.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Copyright</td>
          <td>Copyright</td>
          <td>Copyright</td>
          <td> </td>
          <td>Copyright information for the current resource (or statement that content is in the public domain).</td>
        </tr>
        <tr class="data2">
          <td>License</td>
          <td>License</td>
          <td>Licensing conditions for the current resource.</td>
          <td> </td>
          <td>Data may be distributed under a license. Placing data under a public license (e. g., Creative Commons, GFDL, OpenDocument) while maintaining copyright is recommended. (= DC.Rights.Licence). Example when used with Link: rel="License" href="http://creativecommons.org/licenses/by/2.5/"</td>
        </tr>
        <tr class="data1">
          <td>TermsOfUse</td>
          <td>Terms/use</td>
          <td>Terms of use</td>
          <td> </td>
          <td>Defines conditions under which the data may be analyzed, distributed or changed. "Terms of use" includes concepts like "Usage conditions" and "Specific Restrictions".</td>
        </tr>
        <tr class="data2">
          <td>Disclaimer</td>
          <td>Disclaimer</td>
          <td>Disclaimer</td>
          <td> </td>
          <td>Disclaimer statement, e. g., concerning responsibility for data quality or legal implications.</td>
        </tr>
        <tr class="data1">
          <td>PrivacyPolicy</td>
          <td>Privacy</td>
          <td>Privacy policy</td>
          <td> </td>
          <td>Information and assertion of policy how individual privacy rights are observed.</td>
        </tr>
        <tr class="data2">
          <td>Acknowledgement</td>
          <td>Acknowl.</td>
          <td>Acknowledgement</td>
          <td> </td>
          <td>An acknowledgement of support (e. g., grant money, help, review, permission to reuse published material, etc.)</td>
        </tr>
        <tr class="data1">
          <td>IPRDeclaration</td>
          <td>Declaration</td>
          <td>Declaration</td>
          <td> </td>
          <td>Other forms of IPR declaration not covered by other IPR links (e.g., database rights); also used in cases where an automatic converter can not decide whether a statements is copyright, licence, etc.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="SexStatusEnum" id="SexStatusEnum">SexStatus</a>
    </h3>
    <p>Codes for sex value in humans (clinical status) or animals. The codes are largely based on those defined in DICOM (Digital Imaging and Communications in Medicine, http://medical.nema.org/, Coding Scheme Designator DCM Version 01, PS3.16 Annex B, CID 7455) and ASTM E1633 (= "Standard Specification for Coded Values Used in the Electronic Health Record. Document Number: ASTM E1633-02a. ASTM International, 10-Nov-2002, 76 pages"). Additional codes specific to biology have been added.</p>
    <p>An alternative standard is ISO 5218, which provides only four codes: "0 = Not known, 1 = Male, 2 = Female, 9 = Not specified".  The difference between 0 and 9 is: "(0) implies that the sex of the person is not provided in the personal details i.e. the data has not been supplied and sex cannot be ascertained from the data provided"; "(9) implies that the sex of the person cannot be determined for physical reasons, e. g. a new born baby". ISO 5218 contains fewer and less intuitive codes. For biological purposes many codes would have to be arbitrarily added. G. Hagedorn, 10. August 2004</p>
    <p><strong>States</strong></p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Male</td>
          <td>m</td>
          <td>Male</td>
          <td> </td>
          <td>[= ASTM E1633: M, = ISO 5218: 1]</td>
        </tr>
        <tr class="data2">
          <td>Female</td>
          <td>f</td>
          <td>Female</td>
          <td> </td>
          <td>[= ASTM E1633: F, = ISO 5218: 2]</td>
        </tr>
        <tr class="data1">
          <td>UnknownSex</td>
          <td>?</td>
          <td>Unknown sex</td>
          <td> </td>
          <td>No information regarding the sex is available (= "not recorded" or "not examined"). [= ASTM E1633: U, = ISO 5218: 0]</td>
        </tr>
        <tr class="data2">
          <td>Hermaphrodite</td>
          <td>h</td>
          <td>Hermaphrodite</td>
          <td> </td>
          <td>An organism having both male and female sexual organs at some time during adulthood. General term, not differentiating between simultaneous or sequential hermaphrodites. [= ASTM E1633: H]</td>
        </tr>
        <tr class="data1">
          <td>SimultaneousHermaphrodite</td>
          <td>sh</td>
          <td>Simultaneous hermaphrodite</td>
          <td> </td>
          <td>An organism having both male and female sexual organs at the same time during adulthood. [Not in ASTM E1633; = B in OBIS 1.0 http://www.iobis.org/FAQschema1.shtml]</td>
        </tr>
        <tr class="data2">
          <td>Male2Female</td>
          <td>mc</td>
          <td>Male changing to Female</td>
          <td> </td>
          <td>The organism starts as a male, and changes sex to a female later in life (sequential hermaphrodite: protandry). Examples: seabasses (fish); many plant species; humans that underwent surgical sex change. This terms does not identify a phase in which an individual may be. [= ASTM E1633: MC]</td>
        </tr>
        <tr class="data1">
          <td>Female2Male</td>
          <td>fc</td>
          <td>Female changing to Male</td>
          <td> </td>
          <td>The organism starts as a female, and changes sex to a male later in life (sequential hermaphrodite: protogyny). Example: Wrass reef fishes; some plants; humans that underwent surgical sex change. This terms does not identify a phase in which an individual may be. [= ASTM E1633: FC]</td>
        </tr>
        <tr class="data2">
          <td>HermaphroditeMalePhase</td>
          <td>hm</td>
          <td>Hermaphrodite, male phase</td>
          <td> </td>
          <td>Sequential hermaphrodite in male phase. [Not in ASTM E1633]</td>
        </tr>
        <tr class="data1">
          <td>HermaphroditeFemalePhase</td>
          <td>hf</td>
          <td>Hermaphrodite, female phase</td>
          <td> </td>
          <td>Sequential hermaphrodite in female phase. [Not in ASTM E1633]</td>
        </tr>
        <tr class="data2">
          <td>HermaphroditeTransitional</td>
          <td>ht</td>
          <td>Hermaphrodite, transitional phase</td>
          <td> </td>
          <td>Sequential hermaphrodite currently between sexes. [Not in ASTM E1633, but compare OBIS 1.0 http://www.iobis.org/FAQschema1.shtml]</td>
        </tr>
        <tr class="data1">
          <td>IndeterminateSex</td>
          <td>i</td>
          <td>Indeterminate sex</td>
          <td> </td>
          <td>The organism has been studied, but the sex could not be determined (e.g. in larval forms). Compare "ambiguous" and "unknown" sex.  [perhaps = ISO 5218: 9; perhaps = DICOM: code 121103, 'Undetermined'; compare OBIS 1.0 http://www.iobis.org/FAQschema1.shtml]</td>
        </tr>
        <tr class="data2">
          <td>AmbiguousSex</td>
          <td>a</td>
          <td>Ambiguous sex</td>
          <td> </td>
          <td>The sex organs have been studied, but the result was ambiguous. Includes abnormal mixed sex situations like "gynandromorph" (e. g. an insect is male on one side, female on the other). Compare "indeterminate" and "unknown" sex.  [= ASTM E1633: A]</td>
        </tr>
        <tr class="data1">
          <td>MixedSex</td>
          <td>x</td>
          <td>Mixed sex status</td>
          <td> </td>
          <td>Multiple objects of mixed sex status (situation occurring in collections).</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="BasicSexStatusEnum" id="BasicSexStatusEnum">BasicSexStatus</a>
    </h3>
    <p>Contains basic sex type codes, sufficient for recording human sexes in most administrative contexts (used, e. g., in the Agent type data interface)</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>Male</td>
          <td> </td>
          <td>(see value "Male" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>Female</td>
          <td> </td>
          <td>(see value "Female" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>UnknownSex</td>
          <td> </td>
          <td>(see value "UnknownSex" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="AdditionalSexStatusEnum" id="AdditionalSexStatusEnum">AdditionalSexStatus</a>
    </h3>
    <p>Contains codes in addition to those defined in BasicSexStatusEnum that are necessary for animals and clinical sex descriptions of humans. Additional codes "S, I, HM, HF, HT" has been added to those defined in DICOM /ASTM E1633. On the other side, the DICOM /ASTM E1633 codes "MP = male pseudohermaphrodite" and "FP = female pseudohermaphrodite" are omitted here because they are limited to human sex and express a politically contentious perspective (see http://en.wikipedia.org/wiki/Pseudohermaphrodite). See the UBIF type SexStatusEnum for a union of the enumerated values in this type and those in BasicSexStatusEnum.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>Hermaphrodite</td>
          <td> </td>
          <td>(see value "Hermaphrodite" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>SimultaneousHermaphrodite</td>
          <td> </td>
          <td>(see value "SimultaneousHermaphrodite" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>Male2Female</td>
          <td> </td>
          <td>(see value "Male2Female" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>Female2Male</td>
          <td> </td>
          <td>(see value "Female2Male" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>HermaphroditeMalePhase</td>
          <td> </td>
          <td>(see value "HermaphroditeMalePhase" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>HermaphroditeFemalePhase</td>
          <td> </td>
          <td>(see value "HermaphroditeFemalePhase" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>HermaphroditeTransitional</td>
          <td> </td>
          <td>(see value "HermaphroditeTransitional" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>IndeterminateSex</td>
          <td> </td>
          <td>(see value "IndeterminateSex" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>AmbiguousSex</td>
          <td> </td>
          <td>(see value "AmbiguousSex" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>MixedSex</td>
          <td> </td>
          <td>(see value "MixedSex" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TaxonHierarchyTypeEnum" id="TaxonHierarchyTypeEnum">TaxonHierarchyType</a>
    </h3>
    <p>Defines the type of a taxon hierarchy (list of enumerated values to support application interoperability).</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>UnspecifiedTaxonomy</td>
          <td>Unspecified taxonomy</td>
          <td> </td>
          <td>Conventional taxonomic or nomenclatural arrangement (this could be a PhylogeneticTaxonomy, a NonphylogeneticTaxonomy, or a mixture). In the absences of more specific information about a taxon hierarchy, this may be considered the default value.</td>
        </tr>
        <tr class="data2">
          <td>PhylogeneticTaxonomy</td>
          <td>Phylogenetic taxonomy</td>
          <td> </td>
          <td>Hierarchy resulting from a phylogenetic analysis, regardless of whether the taxonomic conclusions are already implemented or not. This asserts a higher quality of the hierarchy.</td>
        </tr>
        <tr class="data1">
          <td>NonphylogeneticTaxonomy</td>
          <td>Nonphylogenetic taxonomy</td>
          <td> </td>
          <td>A hierarchy explicitly proposing a non-phylogenetic arrangements (herbs, shrubs, small trees, large trees).</td>
        </tr>
        <tr class="data2">
          <td>SubsetFilter</td>
          <td>SubsetFilter</td>
          <td> </td>
          <td>A hierarchy of type "SubsetFilter" is intended only for the purpose of filtering taxa for arbitrary purposes. It will often be a flat list, or a list of filter names, each with a list of taxa. Applications should not offer it as a choice when the user selects a hierarchy for displaying or reporting purposes. Note that conversely, the filter selection dialog in applications should not be restricted to hierarchies of type SubsetFilter. Note, however, that any hierarchy may be used as a filter to define subsets.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="IdentificationCertaintyEnum" id="IdentificationCertaintyEnum">IdentificationCertainty</a>
    </h3>
    <p>Identifications of an Specimen (object/unit) as belonging to a taxon concept may be uncertain. This is especially important in biology, where identification qualifiers like 'cf.' or 'aff.' are often used as part of the scientific name. The following enumerated list provides general categories not restricted to scientific organism names. Note: In biology additional expression is often expressed through the choice of placement of the certainty qualifier. For example, 'Echinonema ferruginea var. campestris' may be qualified as 'cf. Echinonema ferruginea var. campestris', 'Echinonema cf. ferruginea var. campestris', 'Echinonema ferruginea cf. var. campestris'. The first presumably means that the entire name is uncertain, but the infraspecific name may be appropriate, the second indicates that the genus is certain, the species uncertain, and the final that the species in certain and only the infraspecific rank is uncertain. To achieve this level of expressiveness, it is recommended that an additional data element 'IdentificationUncertainTaxonomicRank' of type TaxonomicRankEnum may be combined with an element of IdentificationCertaintyEnum. IdentificationUncertainTaxonomicRank should be optional and omitted to express that an identification is unknown, but the rank not known (e. g. in 'Echinonema ferruginea?'). In ABCD 1.44 a special rank with enumeration beforeName, beforeFirstEpithet, beforeSecondEpithet is used instead.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>IdentificationCertain</td>
          <td>The identification is certain</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>IdentificationUncertain</td>
          <td>The identification is uncertain</td>
          <td> </td>
          <td>In biology this is often expressed with the Latin 'cf.' (confer).</td>
        </tr>
        <tr class="data1">
          <td>IdentificationSimilarTo</td>
          <td>The identification names a similar object class.</td>
          <td> </td>
          <td>In this case the identified object is considered very similar to those objects classified under the given name. Note that in contrast to 'Uncertain' this implies that the object most likely it does not belong to this class. In biology this is often expressed with the Latin 'aff.' or 'afin.' (affinis).</td>
        </tr>
        <tr class="data2">
          <td>IdentificationCertaintyUnknown</td>
          <td>The certainty of identification is unknown.</td>
          <td> </td>
          <td />
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="NomenclaturalTypeStatusOfSpecimensEnum" id="NomenclaturalTypeStatusOfSpecimensEnum">NomenclaturalTypeStatusOfSpecimens</a>
    </h3>
    <p>This list is a first version of a constrained vocabulary to express typifying relations between taxonomic names and specimens (objects/units preserved in collections). Beyond those type categories explicitly governed by nomenclatural codes (Zoology, Botany, Bacterioloy, Virology), the list also includes some additional type status terms. These categories may be helpful when interpreting the original circumscription (topotypes, ex-types), but do not have the same binding status as terms governed by the nomenclatural codes. The enumeration attempts to strike a balance between listing all possible terms, and remaining comprehensible. In general, including too many terms was considered less problematic than omitting terms. Applications may easily select a subset for presentation in their user interface.</p>
    <p>This list is intended as a first version and it is hoped that in the review process through TDWG it will achieve sufficient maturity to be truly useful. It is expected that over time revisions will have to be made. Please use the WIKI (http://wiki.tdwg.org/twiki/bin/view/UBIF/NomenclaturalTypeStatusOfSpecimensDiscussion) to discuss the current list and the lists of synonymous, doubtful, or excluded type terms provided therein.</p>
    <p>Some background information: A type provides the objective standard of reference to determine the application of a taxon name. The type status of a specimen is only meaningful in combination with the name that is being typified (a specimen may have been designated type for multiple names in different publications). The type status of an object may be designated in the original description of a scientific name (original designation), or - under rules layed out in the respective nomenclatural codes - at a later time (subsequent designation). -- For taxa above species rank the type is always a lower rank taxon (e. g., species for genus, genus for family). The type terms for this situation are not included in the enumeration. Ultimately, typication of all taxa goes back to physical type specimens, but this should not be recorded as such in data sets. The indirect type reference in higher taxa means that typification changes to the lower taxon automatically affect the higher taxon.</p>
    <p>The exact definitions of type status differ between nomenclatural codes (ICBN, ICZN, ICNP/ICNB, etc.). The term definitions are intended to be informative and generally applicable across the different codes. The should not be interpreted as authoritative; in nomenclatural work the exact definitions in the respective codes have to be consulted. A duplication of status codes (bot-holo, zoo-holo, bact-holo, etc.) is not considered desirable or necessary. Since the application of the type status terms is constrained by the relationship of the typified name with a specific code, the exact definition can always be unambiguously retrieved.</p>
    <p>The following publications have been consulted to determine the number of type terms that should be included and to prepare the semantic definitions:</p>
    <ul>
              <li>Nomenclatural Glossary for Zoology (January 18 2000; ftp://ftp.york.biosis.org/sysgloss.txt; verified 17. June 2004)</li>
              <li>ICBN St. Louis Code (http://www.bgbm.fu-berlin.de/iapt/nomenclature/code/SaintLouis/0013Ch2Sec2a009.htm; verified 17. June 2004)</li>
              <li>Draft BioCode 4th version (Greuter et al., 1997; http://www.rom.on.ca/biodiversity/biocode/biocode1997.html)</li>
              <li>Glossary of 'type' terminology (Ronald H. Petersen; http://fp.bio.utk.edu/mycology/Nomenclature/nom-type.htm)</li>
              <li>Dictionary of Ichthyology (Brian W. Coad and Don E. McAllister, 2004; http://www.briancoad.com/Dictionary/introduction.htm)</li>
              <li>A useful resource that was not available when writing this proposal might be: Hawksworth, D.L., W.G. Chaloner, O. Krauss, J. McNeill, M.A. Mayo, D.H. Nicolson, P.H.A. Sneath, R.P. Trehane and P.K. Tubbs. 1994. A draft Glossary of terms used in Bionomenclature. (IUBS Monogr. 9) International Union of Biological Sciences, Paris. 74 pp.</li>
            </ul>
   <p>"not a type" was added from the enumeration published in TaxonConceptSchema v 0.8 by J. Kennedy &amp; Robert Kukla in October 2004</p>
    <p>Dr. Miguel A. Alonso-Zarazaga and Dr. Walter Gams are thanked for review and help. - Gregor Hagedorn, 13.7.2004-17.11.2004</p>
    <p><strong>States</strong></p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>allo</td>
          <td>Allotype</td>
          <td> </td>
          <td>A paratype specimen designated from the type series by the original author that is the opposite sex of the holotype. The term is not regulated by the ICZN. [Zoo.]</td>
        </tr>
        <tr class="data2">
          <td>allolecto</td>
          <td>Allolectotype</td>
          <td> </td>
          <td>A paralectotype specimen that is the opposite sex of the lectotype. The term is not regulated by the ICZN. [Zoo.]</td>
        </tr>
        <tr class="data1">
          <td>alloneo</td>
          <td>Alloneotype</td>
          <td> </td>
          <td>A paraneotype specimen that is the opposite sex of the neotype. The term is not regulated by the ICZN. [Zoo.]</td>
        </tr>
        <tr class="data2">
          <td>co</td>
          <td>Cotype</td>
          <td> </td>
          <td>A deprecated term no longer recognized in the ICZN; formerly used for either syntype or paratype [see ICZN Recommendation 73E]. [Zoo.]</td>
        </tr>
        <tr class="data1">
          <td>epi</td>
          <td>Epitype</td>
          <td> </td>
          <td>An epitype is a specimen or illustration selected to serve as an interpretative type when any kind of holotype, lectotype, etc. is demonstrably ambiguous and cannot be critically identified for purposes of the precise application of the name of a taxon (see Art. ICBN 9.7, 9.18). An epitype supplements, rather than replaces existing types. [Bot./Bio.]</td>
        </tr>
        <tr class="data2">
          <td>ex</td>
          <td>Ex-Type</td>
          <td> </td>
          <td>A strain or cultivation derived from some kind of type material. Ex-types are not regulated by the botanical or zoological code. [Zoo./Bot.]</td>
        </tr>
        <tr class="data1">
          <td>exepi</td>
          <td>Ex-Epitype</td>
          <td> </td>
          <td>A strain or cultivation derived from epitype material. Ex-types are not regulated by the botanical or zoological code. [Bot.]</td>
        </tr>
        <tr class="data2">
          <td>exholo</td>
          <td>Ex-Holotype</td>
          <td> </td>
          <td>A strain or cultivation derived from holotype material. Ex-types are not regulated by the botanical or zoological code. [Zoo./Bot.]</td>
        </tr>
        <tr class="data1">
          <td>exiso</td>
          <td>Ex-Isotype</td>
          <td> </td>
          <td>A strain or cultivation derived from isotype material. Ex-types are not regulated by the botanical or zoological code. [Zoo./Bot.]</td>
        </tr>
        <tr class="data2">
          <td>exlecto</td>
          <td>Ex-Lectotype</td>
          <td> </td>
          <td>A strain or cultivation derived from lectotype material. Ex-types are not regulated by the botanical or zoological code. [Zoo./Bot.]</td>
        </tr>
        <tr class="data1">
          <td>exneo</td>
          <td>Ex-Neotype</td>
          <td> </td>
          <td>A strain or cultivation derived from neotype material. Ex-types are not regulated by the botanical or zoological code. [Zoo./Bot.]</td>
        </tr>
        <tr class="data2">
          <td>expara</td>
          <td>Ex-Paratype</td>
          <td> </td>
          <td>A strain or cultivation derived from paratype material. Ex-types are not regulated by the botanical or zoological code. [Zoo./Bot.]</td>
        </tr>
        <tr class="data1">
          <td>exsyn</td>
          <td>Ex-Syntype</td>
          <td> </td>
          <td>A strain or cultivation derived from neotype material. Ex-types are not regulated by the botanical or zoological code. [Zoo./Bot.]</td>
        </tr>
        <tr class="data2">
          <td>hapanto</td>
          <td>Hapantotype</td>
          <td> </td>
          <td>One or more preparations of directly related individuals representing distinct stages in the life cycle, which together form the type in an extant species of protistan [ICZN Article 72.5.4]. A hapantotype, while a series of individuals, is a holotype that must not be restricted by lectotype selection. If a hapantotype is found to contain individuals of more than one species, however, components may be excluded until it contains individuals of only one species [ICZN Article 73.3.2]. [Zoo.]</td>
        </tr>
        <tr class="data1">
          <td>holo</td>
          <td>Holotype</td>
          <td> </td>
          <td>The one specimen or other element used or designated by the original author at the time of publication of the original description as the nomenclatural type of a species or infraspecific taxon. A holotype may be 'explicit' if it is clearly stated in the originating publication or 'implicit' if it is the single specimen proved to have been in the hands of the originating author when the description was published. [Zoo./Bot./Bio.]</td>
        </tr>
        <tr class="data2">
          <td>icono</td>
          <td>Iconotype</td>
          <td> </td>
          <td>A drawing or photograph (also called 'phototype') of a type specimen. Note: the term "iconotype" is not used in the ICBN, but implicit in, e. g., ICBN Art. 7 and 38. [Zoo./Bot.]</td>
        </tr>
        <tr class="data1">
          <td>iso</td>
          <td>Isotype</td>
          <td> </td>
          <td>An isotype is any duplicate of the holotype (i. e. part of a single gathering made by a collector at one time, from which the holotype was derived); it is always a specimen (ICBN Art. 7). [Bot.]</td>
        </tr>
        <tr class="data2">
          <td>isolecto</td>
          <td>Isolectotype</td>
          <td> </td>
          <td>A duplicate of a neotype, compare lectotype. [Bot.]</td>
        </tr>
        <tr class="data1">
          <td>isoneo</td>
          <td>Isoneotype</td>
          <td> </td>
          <td>A duplicate of a neotype, compare neotype. [Bot.]</td>
        </tr>
        <tr class="data2">
          <td>isosyn</td>
          <td>Isosyntype</td>
          <td> </td>
          <td>A duplicate of a syntype, compare isotype = duplicate of holotype. [Bot.]</td>
        </tr>
        <tr class="data1">
          <td>lecto</td>
          <td>Lectotype</td>
          <td> </td>
          <td>A specimen or other element designated subsequent to the publication of the original description from the original material (syntypes or paratypes) to serve as nomenclatural type. Lectotype designation can occur only where no holotype was designated at the time of publication or if it is missing (ICBN Art. 7, ICZN Art. 74). [Zoo./Bot.] -- Note: the BioCode defines lectotype as selection from holotype material in cases where the holotype material contains more than one taxon [Bio.].</td>
        </tr>
        <tr class="data2">
          <td>neo</td>
          <td>Neotype</td>
          <td> </td>
          <td>A specimen designated as nomenclatural type subsequent to the publication of the original description in cases where the original holotype, lectotype, all paratypes and syntypes are lost or destroyed, or suppressed by the (botanical or zoological) commission on nomenclature. In zoology also called "Standard specimen" or "Representative specimen". [Zoo./Bot./Bio.]</td>
        </tr>
        <tr class="data1">
          <td>para</td>
          <td>Paratype</td>
          <td> </td>
          <td>All of the specimens in the type series of a species or infraspecific taxon other than the holotype (and, in botany, isotypes). Paratypes must have been at the disposition of the author at the time when the original description was created and must have been designated and indicated in the publication. Judgment must be exercised on paratype status, for only rarely are specimens explicitly cited as paratypes, but usually as "specimens examined," "other material seen", etc. [Zoo./Bot.]</td>
        </tr>
        <tr class="data2">
          <td>paralecto</td>
          <td>Paralectotype</td>
          <td> </td>
          <td>All of the specimens in the syntype series of a species or infraspecific taxon other than the lectotype itself. Also called "lectoparatype". [Zoo.]</td>
        </tr>
        <tr class="data1">
          <td>paraneo</td>
          <td>Paraneotype</td>
          <td> </td>
          <td>All of the specimens in the syntype series of a species or infraspecific taxon other than the neotype itself. Also called "neoparatype". [Zoo.]</td>
        </tr>
        <tr class="data2">
          <td>plasto</td>
          <td>Plastotype</td>
          <td> </td>
          <td>A copy or cast of type material, esp. relevant for fossil types. Not regulated by the botanical or zoological code (?). [Zoo./Bot.]</td>
        </tr>
        <tr class="data1">
          <td>plastoholo</td>
          <td>Plastoholotype</td>
          <td> </td>
          <td>A copy or cast of holotype material (compare Plastotype).</td>
        </tr>
        <tr class="data2">
          <td>plastoiso</td>
          <td>Plastoisotype</td>
          <td> </td>
          <td>A copy or cast of isotype material (compare Plastotype).</td>
        </tr>
        <tr class="data1">
          <td>plastolecto</td>
          <td>Plastolectotype</td>
          <td> </td>
          <td>A copy or cast of lectotype material (compare Plastotype).</td>
        </tr>
        <tr class="data2">
          <td>plastoneo</td>
          <td>Plastoneotype</td>
          <td> </td>
          <td>A copy or cast of neotype material (compare Plastotype).</td>
        </tr>
        <tr class="data1">
          <td>plastopara</td>
          <td>Plastoparatype</td>
          <td> </td>
          <td>A copy or cast of paratype material (compare Plastotype).</td>
        </tr>
        <tr class="data2">
          <td>plastosyn</td>
          <td>Plastosyntype</td>
          <td> </td>
          <td>A copy or cast of syntype material (compare Plastotype).</td>
        </tr>
        <tr class="data1">
          <td>syn</td>
          <td>Syntypes</td>
          <td> </td>
          <td>The series of specimens used to describe a species or infraspecific taxon when neither a single holotype by the original author, nor a lectotype in a subsequent publication has been designated. The syntypes collectively constitute the name-bearing type. [Zoo./Bot.]</td>
        </tr>
        <tr class="data2">
          <td>topo</td>
          <td>Topotype</td>
          <td> </td>
          <td>One or more specimens collected at the same location as the type series (type locality), regardless of whether they are part of the type series. Topotypes are not regulated by the botanical or zoological code. Also called "locotype". [Zoo./Bot.]</td>
        </tr>
        <tr class="data1">
          <td>type</td>
          <td>Type</td>
          <td> </td>
          <td>a) A specimen designated or indicated any kind of type of a species or infraspecific taxon. If possible more specific type terms (holotype, syntype, etc.) should be applied. b) the type name of a name of higher rank for taxa above the species rank. [General]</td>
        </tr>
        <tr class="data2">
          <td>sec</td>
          <td>Secondary type</td>
          <td> </td>
          <td>A referred, described, measured or figured specimen in the original publication (including a neo/lectotypification publication) that is not a primary type. [General]</td>
        </tr>
        <tr class="data1">
          <td>supp</td>
          <td>Supplementary type</td>
          <td> </td>
          <td>A referred, described, measured or figured specimen in a revision of a previously described taxon. This is not formally considered a type. Provided as an alternative to "not a type" to express knowledge about material elsewhere presumed to have a more formal type status. [General]</td>
        </tr>
        <tr class="data2">
          <td>no</td>
          <td>not a type</td>
          <td> </td>
          <td>For specimens erroneously labelled as types an explicit negative statement may be desirable. [General]</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="NomenclaturalStatusEnum" id="NomenclaturalStatusEnum">NomenclaturalStatus</a>
    </h3>
    <p>Controlled vocabulary expressing nomenclatural status of a biological taxon name. ### This needs urgently revision! Enumeration of possible values for nomenclatural status. (Source: initial LinneanCore.)</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>Rejected</td>
          <td>Rejected</td>
          <td> </td>
          <td>The name is rejected (invalid, not accepted)</td>
        </tr>
        <tr class="data2">
          <td>Fails</td>
          <td>Fails</td>
          <td> </td>
          <td>The name fails the requirement test for an acceptable name (e.g. pre-starting-date names).</td>
        </tr>
        <tr class="data1">
          <td>Conserved</td>
          <td>Conserved</td>
          <td> </td>
          <td>The name is accepted by means of an explicit conservation proposal</td>
        </tr>
        <tr class="data2">
          <td>Sanctioned</td>
          <td>Sanctioned</td>
          <td> </td>
          <td>The name is sanctioned (a special form of automatic conservation used in ICBN for fungi)</td>
        </tr>
        <tr class="data1">
          <td>Accepted</td>
          <td>Accepted</td>
          <td> </td>
          <td>The name is accepted.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="NomenclaturalCodesEnum" id="NomenclaturalCodesEnum">NomenclaturalCodes</a>
    </h3>
    <p>Enumeration of nomenclatural codes under which a name is considered valid. (Source: comparison of enumerations in ABCD 1.49 and first LinneanCore draft.) - Gregor Hagedorn</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>Viral</td>
          <td> </td>
          <td>(see value "Viral" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>Bacteriological</td>
          <td> </td>
          <td>(see value "Bacteriological" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td>Botanical</td>
          <td>International Code of Botanical Nomenclature, ICBN</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>Zoological</td>
          <td>International Code of Zoological Nomenclature, ICZN</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>CultivatedPlant</td>
          <td>International Code of cultivated plants, ICNCP</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>Indeterminate</td>
          <td>Examples: ambiregnally named taxa can not be fixed to a code; to express the (pre-starting point) names on which the sanctioning mechanism of fungal names is based.</td>
          <td> </td>
          <td />
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TaxonomicRankEnum" id="TaxonomicRankEnum">TaxonomicRank</a>
    </h3>
    <p>Enumerated codes to express the rank of a taxon (scientific organism name) in a taxonomic hierarchy. The list is intended to be interoperable between name providers for bacteria, viruses, fungi, plants, and animals. It is not assumed that in each taxonomic group all ranks have to be used. Individual applications may select appropriate subsets (which may be based on information given inside the enumerated values, see Specifications/BioCode-, Botany-, Zoology-, and BacteriaStatus). The enumeration attempts to strike a balance between listing all possible rank terms, and remaining comprehensible. For example, the "infra-" ranks specifically mentioned in BioCode have been included (although very rarely used), but the additional intermediate zoological ranks (micro, nano, pico, etc.) are not included. Whether the selection of infraspecific ranks (some informal ranks, esp. from bacteriology, may be missing!) probably needs some discussion. However, it is believed that this list may help to start developing data sets that can easily be integrated across the barriers of language and taxonomic traditions.</p>
    <p>Not included in the list are the botanical "notho-" ranks, which are used to designate hybrids (nothospecies, nothogenus). It is assumed they can be generated from separate information that the taxon is a hybrid. ICBN §4.4 states: "The subordinate ranks of nothotaxa are the same as the subordinate ranks of non-hybrid taxa, except that nothogenus is the highest rank permitted".</p>
    <p>The following publications have been consulted to determine the number of type terms that should be included and to prepare the semantic definitions:</p>
    <ul>
              <li>The Berlin Taxonomic Information Model, MoReTax view (Berendsohn &amp; al., http://www.bgbm.org/scripts/ASP/BGBMModel/Catalogues.asp?Cat=MT</li>
              <li>DiversityTaxonomy model version 0.7 (G. Hagedorn &amp; T. Gräfenhan 2002, http://www.diversitycampus.net/Workbench/Taxonomy/Model/InformationModels.html)</li>
              <li>ABCD version 1.44, types HigherTaxonRankType and RankAbbreviationType, by W. Berendsohn, reviewed by D. Hobern</li>
              <li>TaxCat2 - Database of Botanical Taxonomic Categories by Jörg Ochsmann, IPK Gatersleben; http://mansfeld.ipk-gatersleben.de/TaxCat2/default.htm</li>
            </ul>
    <p>Many thanks for review and help go to Dr. Walter Gams. - Gregor Hagedorn, 13.7.2004-17.11.2004.</p>
    <p>Note: the list of all ranks is implemented as a union of all following rank subsets. Note that although BioCode has been used to define the partition into subsets, the ranks are not limited to BioCode but should be an interoperable superset of ranks used in Virology, Bacteriology, Botany and Zoology.</p>
    <p>Technical note: It would be preferable to define the values in separate types and define this type as xs:union memberTypes="TaxonomicRankBelowSubspeciesEnum TaxonomicRankSpeciesGroupEnum TaxonomicRankGenusSubdivisionEnum TaxonomicRankGenusGroupEnum TaxonomicRankFamilySubdivisionEnum TaxonomicRankFamilyGroupEnum TaxonomicRankAboveSuperfamilyEnum". However, as explained in the annotation of AgentRoleEnum, Stylus Studio and Xerces have problems with type derivations involving union types. Therefore the current work-around was chosen.</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Abbrev.</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td>any_rank</td>
          <td>unknown</td>
          <td>tax. of unknown or undefined rank</td>
          <td> </td>
          <td>Use this if no information exists or can be deduced. If at least the rank group is known other values ("any_infrasp", "any_infragen", "any_supragen") should be preferred.</td>
        </tr>
        <tr class="data2">
          <td>any_infrasp</td>
          <td>tax. infrasp.</td>
          <td>infraspecific tax. of undefined rank</td>
          <td> </td>
          <td>Undefined ranks (using either no rank identifier in botany, or using greek letters or symbols like stars, crosses) occur in very old publications. Most frequently these are to be interpreted as varieties, but occasionally they are forms or subspecies (see Stearn, W.T. 1957: Species plantarum (Facsimile); Introduction. 1. London, p. 90-95, 160-161, 163). The interpretation of these cases requires taxonomic knowledge that may not be available at the time when data are parsed. Such lack of knowledge can be expressed using this rank identifier.</td>
        </tr>
        <tr class="data1">
          <td>cand</td>
          <td>cand.</td>
          <td>candidate</td>
          <td> </td>
          <td>'Candidatus' rank is proposed in bacteriology for putative taxa, which could not yet be studied sufficiently to warrant the creation of a name with a known rank. (Murray, R.G.E. &amp; Schleifer, K.H.: Taxonomic notes: a proposal for recording the properties of putative taxa of procaryotes. Int. J. Syst. Bacteriol., 1994, 44, 174-176).</td>
        </tr>
        <tr class="data2">
          <td>fsp</td>
          <td>f. sp.</td>
          <td>special form</td>
          <td> </td>
          <td>The ICBN does not formally cover formae specialis (art. 4, note 3). However, because of the economic importance of pathogenic f. sp., and since it is common practice to handle them as if the code would apply (i. e. priority usually observed, name quoted with author), they are included here.</td>
        </tr>
        <tr class="data1">
          <td>subsubfm</td>
          <td>subsubfm.</td>
          <td>subsubform</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>subfm</td>
          <td>subfm.</td>
          <td>subform</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>fm</td>
          <td>fm.</td>
          <td>form</td>
          <td> </td>
          <td>Form, race, variety are not subject to regulation in zoology; see ICZN Article 1.3.4</td>
        </tr>
        <tr class="data2">
          <td>subsubvar</td>
          <td>subsubvar.</td>
          <td>sub-sub-variety</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>subvar</td>
          <td>subvar.</td>
          <td>sub-variety</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>var</td>
          <td>var.</td>
          <td>variety</td>
          <td> </td>
          <td>Form, race, variety are not subject to regulation in zoology; see ICZN Article 1.3.4 Examples: Pinus nigra var. caramanica (= "P. nigra subsp. nigra var. caramanica"; Taxus baccata var. variegata</td>
        </tr>
        <tr class="data1">
          <td>pv</td>
          <td>pathovar.</td>
          <td>patho-variety</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>bv</td>
          <td>biovar.</td>
          <td>bio-variety</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>cv</td>
          <td>cult.</td>
          <td>cultivar</td>
          <td> </td>
          <td>The epithet is usually output in single quotes and may contain multiple words, see ICBN §28. Examples: Taxus baccata 'Variegata', Juniperus ×pfitzeriana 'Wilhelm Pfitzer'; Magnolia 'Elizabeth' (= a hybrid, no species epithet).</td>
        </tr>
        <tr class="data2">
          <td>convar</td>
          <td>convar.</td>
          <td>convar</td>
          <td> </td>
          <td>Used in cultivated plants (ICNCP), but deprecated, see 'Some notes on problems of taxonomy and nomenclature of cultivated plants' by J. Ochsmann, http://www.genres.de/IGRREIHE/IGRREIHE/DDD/22-08.pdf</td>
        </tr>
        <tr class="data1">
          <td>cvgroup</td>
          <td>cultivar. group</td>
          <td>cultivar-group</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>graftchimaera</td>
          <td>graft-chimaera</td>
          <td>graft-chimaera</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>infrasp</td>
          <td>infrasp.</td>
          <td>infraspecies</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>subsp_aggr</td>
          <td>aggr.</td>
          <td>subspecific aggregate (= group, complex)</td>
          <td> </td>
          <td>A loosely defined group of subspecies. Zoology: "Aggregate - a group of  subspecies within a species. An aggregate may be denoted by a group name interpolated in parentheses."</td>
        </tr>
        <tr class="data1">
          <td>ssp</td>
          <td>ssp.</td>
          <td>subspecies</td>
          <td> </td>
          <td>Examples: Pinus nigra subsp. nigra Homo sapiens sapiens</td>
        </tr>
        <tr class="data2">
          <td>sp</td>
          <td>sp.</td>
          <td>species</td>
          <td> </td>
          <td>Examples: Taxus baccata,  Homo sapiens</td>
        </tr>
        <tr class="data1">
          <td>aggr</td>
          <td>aggr.</td>
          <td>species aggregate (= species group, species complex)</td>
          <td> </td>
          <td>A loosely defined group of species. Zoology: "Aggregate - a group of species, other than a subgenus, within a genus. An aggregate may be denoted by a group name interpolated in parentheses." -- The Berlin/MoreTax model notes: "[these] aren't taxonomic ranks but cirumscriptions because on the one hand they are necessary for the concatenation of the fullname and on the other hand they are necessary for distinguishing the aggregate or species group from the microspecies." Compare subspecific aggregate for a group of subspecies within a species!</td>
        </tr>
        <tr class="data2">
          <td>any_infragen</td>
          <td>tax. infragen.</td>
          <td>infrageneric tax. of undefined rank</td>
          <td> </td>
          <td>A name that appear between a genus name and a species epitheton and is not clearly marked as series or section, or other may be assigned to this rank until the true rank can be assigned by a taxonomic expert.</td>
        </tr>
        <tr class="data1">
          <td>subser</td>
          <td>subser.</td>
          <td>subseries</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>ser</td>
          <td>ser.</td>
          <td>series</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>subsect</td>
          <td>subsect.</td>
          <td>subsection</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>sect</td>
          <td>sect.</td>
          <td>section</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>infragen</td>
          <td>infragen.</td>
          <td>infragenus</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>subgen</td>
          <td>subgen.</td>
          <td>subgenus</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>gen</td>
          <td>gen.</td>
          <td>genus</td>
          <td> </td>
          <td>Examples: Magnolia Homo</td>
        </tr>
        <tr class="data2">
          <td>infratrib</td>
          <td>infratrib.</td>
          <td>infratribe</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>subtrib</td>
          <td>subtrib.</td>
          <td>subtribe</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>trib</td>
          <td>trib.</td>
          <td>tribe</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>supertrib</td>
          <td>supertrib.</td>
          <td>supertribe</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>infrafam</td>
          <td>infrafam.</td>
          <td>infrafamily</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>subfam</td>
          <td>subfam.</td>
          <td>subfamily</td>
          <td> </td>
          <td>Examples: Magnolioideae</td>
        </tr>
        <tr class="data2">
          <td>fam</td>
          <td>fam.</td>
          <td>family</td>
          <td> </td>
          <td>Examples: Magnoliaceae Hominidae</td>
        </tr>
        <tr class="data1">
          <td>superfam</td>
          <td>superfam.</td>
          <td>superfamily</td>
          <td> </td>
          <td>Examples: Magnoliacea</td>
        </tr>
        <tr class="data2">
          <td>infraord</td>
          <td>infraord.</td>
          <td>infraorder</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>subord</td>
          <td>subord.</td>
          <td>suborder</td>
          <td> </td>
          <td>Examples: Magnolineae Catarrhini</td>
        </tr>
        <tr class="data2">
          <td>ord</td>
          <td>ord.</td>
          <td>order </td>
          <td> </td>
          <td>Examples: Magnoliales Primates</td>
        </tr>
        <tr class="data1">
          <td>superord</td>
          <td>superord.</td>
          <td>superorder</td>
          <td> </td>
          <td>Examples: Magnolianae</td>
        </tr>
        <tr class="data2">
          <td>infracl</td>
          <td>infracl.</td>
          <td>infraclass</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>subcl</td>
          <td>subcl.</td>
          <td>subclass </td>
          <td> </td>
          <td>Examples: Magnoliidae Eutheria</td>
        </tr>
        <tr class="data2">
          <td>cl</td>
          <td>cl.</td>
          <td>class </td>
          <td> </td>
          <td>Examples: Magnoliopsida Mammalia</td>
        </tr>
        <tr class="data1">
          <td>supercl</td>
          <td>supercl.</td>
          <td>superclass</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>infraphyl_div</td>
          <td>infraphyl./div.</td>
          <td>infraphylum (= infradivision)</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>subphyl_div</td>
          <td>subphyl./div.</td>
          <td>subphylum (= subdivision)</td>
          <td> </td>
          <td>Examples: Magnoliophytina Vertebrata</td>
        </tr>
        <tr class="data2">
          <td>phyl_div</td>
          <td>phyl./div.</td>
          <td>phylum (= division)</td>
          <td> </td>
          <td>Examples: Magnoliophyta Chordata</td>
        </tr>
        <tr class="data1">
          <td>superphyl_div</td>
          <td>superphyl./div.</td>
          <td>superphylum (= superdivision)</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>infrareg</td>
          <td>infrareg.</td>
          <td>infrakingdom</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data1">
          <td>subreg</td>
          <td>subreg.</td>
          <td>subkingdom</td>
          <td> </td>
          <td />
        </tr>
        <tr class="data2">
          <td>reg</td>
          <td>reg.</td>
          <td>kingdom</td>
          <td> </td>
          <td>Examples: Plantae Animalia</td>
        </tr>
        <tr class="data1">
          <td>superreg</td>
          <td>superreg.</td>
          <td>super kingdom</td>
          <td> </td>
          <td>Examples: Eucaryota</td>
        </tr>
        <tr class="data2">
          <td>dom</td>
          <td>dom.</td>
          <td>domain (= empire)</td>
          <td> </td>
          <td>Examples: Archaea (= Archaeobacteria), Bacteria (= Eubacteria), Eukarya (= Eukaryota)</td>
        </tr>
        <tr class="data1">
          <td>any_supragen</td>
          <td>tax. supragen.</td>
          <td>suprageneric tax. of undefined rank</td>
          <td> </td>
          <td>This value indicates that the rank of a name is unknown. Compare "incertae sedis" which is commonly used as a replacement for a taxon to group all taxa whose position in the classification or phylogenetic tree is uncertain.</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TaxonomicRankBelowSubspeciesEnum" id="TaxonomicRankBelowSubspeciesEnum">TaxonomicRankBelowSubspecies</a>
    </h3>
    <p>Subset of ranks; equivalent to BioCode "infra-subspecfic", i.e. below the species group</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>any_infrasp</td>
          <td> </td>
          <td>(see value "any_infrasp" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>cand</td>
          <td> </td>
          <td>(see value "cand" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>fsp</td>
          <td> </td>
          <td>(see value "fsp" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>subsubfm</td>
          <td> </td>
          <td>(see value "subsubfm" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>subfm</td>
          <td> </td>
          <td>(see value "subfm" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>fm</td>
          <td> </td>
          <td>(see value "fm" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>subsubvar</td>
          <td> </td>
          <td>(see value "subsubvar" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>subvar</td>
          <td> </td>
          <td>(see value "subvar" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>var</td>
          <td> </td>
          <td>(see value "var" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>pv</td>
          <td> </td>
          <td>(see value "pv" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>bv</td>
          <td> </td>
          <td>(see value "bv" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>cv</td>
          <td> </td>
          <td>(see value "cv" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>convar</td>
          <td> </td>
          <td>(see value "convar" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>cvgroup</td>
          <td> </td>
          <td>(see value "cvgroup" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>graftchimaera</td>
          <td> </td>
          <td>(see value "graftchimaera" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>infrasp</td>
          <td> </td>
          <td>(see value "infrasp" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TaxonomicRankSpeciesGroupEnum" id="TaxonomicRankSpeciesGroupEnum">TaxonomicRankSpeciesGroup</a>
    </h3>
    <p>Subset of ranks; equivalent to BioCode "species group", i.e. only species and subspecies</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>subsp_aggr</td>
          <td> </td>
          <td>(see value "subsp_aggr" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>ssp</td>
          <td> </td>
          <td>(see value "ssp" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>sp</td>
          <td> </td>
          <td>(see value "sp" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TaxonomicRankGenusSubdivisionEnum" id="TaxonomicRankGenusSubdivisionEnum">TaxonomicRankGenusSubdivision</a>
    </h3>
    <p>Subset of ranks; equivalent to BioCode ""subdivision of a genus" ", i.e. all ranks between genus and species group (i.e. not including subgenus and species)</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>aggr</td>
          <td> </td>
          <td>(see value "aggr" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>any_infragen</td>
          <td> </td>
          <td>(see value "any_infragen" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>subser</td>
          <td> </td>
          <td>(see value "subser" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>ser</td>
          <td> </td>
          <td>(see value "ser" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>subsect</td>
          <td> </td>
          <td>(see value "subsect" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>sect</td>
          <td> </td>
          <td>(see value "sect" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TaxonomicRankGenusGroupEnum" id="TaxonomicRankGenusGroupEnum">TaxonomicRankGenusGroup</a>
    </h3>
    <p>Subset of ranks; equivalent to BioCode "genus group", i.e. infragenus to genus</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>infragen</td>
          <td> </td>
          <td>(see value "infragen" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>subgen</td>
          <td> </td>
          <td>(see value "subgen" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>gen</td>
          <td> </td>
          <td>(see value "gen" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TaxonomicRankFamilySubdivisionEnum" id="TaxonomicRankFamilySubdivisionEnum">TaxonomicRankFamilySubdivision</a>
    </h3>
    <p>Subset of ranks; equivalent to BioCode "subdivision of a family", i.e. ranks between genus group and family group</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>infratrib</td>
          <td> </td>
          <td>(see value "infratrib" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>subtrib</td>
          <td> </td>
          <td>(see value "subtrib" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>trib</td>
          <td> </td>
          <td>(see value "trib" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>supertrib</td>
          <td> </td>
          <td>(see value "supertrib" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TaxonomicRankFamilyGroupEnum" id="TaxonomicRankFamilyGroupEnum">TaxonomicRankFamilyGroup</a>
    </h3>
    <p>Subset of ranks; equivalent to BioCode "family group", i.e. infrafamily to superfamily</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>infrafam</td>
          <td> </td>
          <td>(see value "infrafam" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>subfam</td>
          <td> </td>
          <td>(see value "subfam" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>fam</td>
          <td> </td>
          <td>(see value "fam" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>superfam</td>
          <td> </td>
          <td>(see value "superfam" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <h3>
      <a name="TaxonomicRankAboveSuperfamilyEnum" id="TaxonomicRankAboveSuperfamilyEnum">TaxonomicRankAboveSuperfamily</a>
    </h3>
    <p>Subset of ranks; equivalent to BioCode "suprafamilial". This rank group includes all ranks higher than superfamily (class, phylum/division, kingdom, domain)</p>
    <p>
      <strong>States</strong>
    </p>
    <table width="100%" border="1" cellspacing="0" cellpadding="3" frame="hsides" rules="groups" summary="item table'">
      <thead>
        <tr class="header">
          <th>Value</th>
          <th>Label</th>
          <th> </th>
          <th>Description</th>
        </tr>
      </thead>
      <tbody>
        <tr class="data1">
          <td />
          <td>infraord</td>
          <td> </td>
          <td>(see value "infraord" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>subord</td>
          <td> </td>
          <td>(see value "subord" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>ord</td>
          <td> </td>
          <td>(see value "ord" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>superord</td>
          <td> </td>
          <td>(see value "superord" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>infracl</td>
          <td> </td>
          <td>(see value "infracl" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>subcl</td>
          <td> </td>
          <td>(see value "subcl" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>cl</td>
          <td> </td>
          <td>(see value "cl" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>supercl</td>
          <td> </td>
          <td>(see value "supercl" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>infraphyl_div</td>
          <td> </td>
          <td>(see value "infraphyl_div" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>subphyl_div</td>
          <td> </td>
          <td>(see value "subphyl_div" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>phyl_div</td>
          <td> </td>
          <td>(see value "phyl_div" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>superphyl_div</td>
          <td> </td>
          <td>(see value "superphyl_div" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>infrareg</td>
          <td> </td>
          <td>(see value "infrareg" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>subreg</td>
          <td> </td>
          <td>(see value "subreg" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>reg</td>
          <td> </td>
          <td>(see value "reg" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>superreg</td>
          <td> </td>
          <td>(see value "superreg" defined elsewhere)</td>
        </tr>
        <tr class="data1">
          <td />
          <td>dom</td>
          <td> </td>
          <td>(see value "dom" defined elsewhere)</td>
        </tr>
        <tr class="data2">
          <td />
          <td>any_supragen</td>
          <td> </td>
          <td>(see value "any_supragen" defined elsewhere)</td>
        </tr>
      </tbody>
    </table>
    <p class="small">
      <a href="#toc">(Return to table of contents)</a>
      <br /> </p>
    <hr />
  </body>
</html>
